WARNING: Error loading config file: /home/.docker/config.json: Invalid auth configuration file
-- Logs begin at Tue 2021-08-17 17:50:16 UTC, end at Tue 2021-08-17 17:51:29 UTC. --
Aug 17 17:50:16 config-control-plane systemd[1]: Condition check resulted in kubelet: The Kubernetes Node Agent being skipped.
Aug 17 17:50:21 config-control-plane systemd[1]: Starting kubelet: The Kubernetes Node Agent...
Aug 17 17:50:21 config-control-plane systemd[1]: Started kubelet: The Kubernetes Node Agent.
Aug 17 17:50:21 config-control-plane kubelet[286]: Flag --fail-swap-on has been deprecated, This parameter should be set via the config file specified by the Kubelet's --config flag. See https://kubernetes.io/docs/tasks/administer-cluster/kubelet-config-file/ for more information.
Aug 17 17:50:21 config-control-plane kubelet[286]: Flag --provider-id has been deprecated, This parameter should be set via the config file specified by the Kubelet's --config flag. See https://kubernetes.io/docs/tasks/administer-cluster/kubelet-config-file/ for more information.
Aug 17 17:50:21 config-control-plane kubelet[286]: Flag --fail-swap-on has been deprecated, This parameter should be set via the config file specified by the Kubelet's --config flag. See https://kubernetes.io/docs/tasks/administer-cluster/kubelet-config-file/ for more information.
Aug 17 17:50:21 config-control-plane kubelet[286]: Flag --cgroup-root has been deprecated, This parameter should be set via the config file specified by the Kubelet's --config flag. See https://kubernetes.io/docs/tasks/administer-cluster/kubelet-config-file/ for more information.
Aug 17 17:50:21 config-control-plane kubelet[286]: I0817 17:50:21.568248     286 server.go:197] "Warning: For remote container runtime, --pod-infra-container-image is ignored in kubelet, which should be set in that remote runtime instead"
Aug 17 17:50:21 config-control-plane kubelet[286]: Flag --fail-swap-on has been deprecated, This parameter should be set via the config file specified by the Kubelet's --config flag. See https://kubernetes.io/docs/tasks/administer-cluster/kubelet-config-file/ for more information.
Aug 17 17:50:21 config-control-plane kubelet[286]: Flag --provider-id has been deprecated, This parameter should be set via the config file specified by the Kubelet's --config flag. See https://kubernetes.io/docs/tasks/administer-cluster/kubelet-config-file/ for more information.
Aug 17 17:50:21 config-control-plane kubelet[286]: Flag --fail-swap-on has been deprecated, This parameter should be set via the config file specified by the Kubelet's --config flag. See https://kubernetes.io/docs/tasks/administer-cluster/kubelet-config-file/ for more information.
Aug 17 17:50:21 config-control-plane kubelet[286]: Flag --cgroup-root has been deprecated, This parameter should be set via the config file specified by the Kubelet's --config flag. See https://kubernetes.io/docs/tasks/administer-cluster/kubelet-config-file/ for more information.
Aug 17 17:50:21 config-control-plane kubelet[286]: I0817 17:50:21.897592     286 server.go:440] "Kubelet version" kubeletVersion="v1.21.1"
Aug 17 17:50:21 config-control-plane kubelet[286]: I0817 17:50:21.897930     286 server.go:851] "Client rotation is on, will bootstrap in background"
Aug 17 17:50:21 config-control-plane kubelet[286]: I0817 17:50:21.901736     286 dynamic_cafile_content.go:167] Starting client-ca-bundle::/etc/kubernetes/pki/ca.crt
Aug 17 17:50:21 config-control-plane kubelet[286]: E0817 17:50:21.911881     286 certificate_manager.go:437] Failed while requesting a signed certificate from the master: cannot create certificate signing request: Post "https://config-control-plane:6443/apis/certificates.k8s.io/v1/certificatesigningrequests": dial tcp 172.18.0.3:6443: connect: connection refused
Aug 17 17:50:23 config-control-plane kubelet[286]: E0817 17:50:23.918738     286 certificate_manager.go:437] Failed while requesting a signed certificate from the master: cannot create certificate signing request: Post "https://config-control-plane:6443/apis/certificates.k8s.io/v1/certificatesigningrequests": dial tcp 172.18.0.3:6443: connect: connection refused
Aug 17 17:50:26 config-control-plane kubelet[286]: W0817 17:50:26.912367     286 sysinfo.go:203] Nodes topology is not available, providing CPU topology
Aug 17 17:50:26 config-control-plane kubelet[286]: I0817 17:50:26.918227     286 container_manager_linux.go:278] "Container manager verified user specified cgroup-root exists" cgroupRoot=[kubelet]
Aug 17 17:50:26 config-control-plane kubelet[286]: I0817 17:50:26.918313     286 container_manager_linux.go:283] "Creating Container Manager object based on Node Config" nodeConfig={RuntimeCgroupsName: SystemCgroupsName: KubeletCgroupsName: ContainerRuntime:remote CgroupsPerQOS:true CgroupRoot:/kubelet CgroupDriver:cgroupfs KubeletRootDir:/var/lib/kubelet ProtectKernelDefaults:false NodeAllocatableConfig:{KubeReservedCgroupName: SystemReservedCgroupName: ReservedSystemCPUs: EnforceNodeAllocatable:map[pods:{}] KubeReserved:map[] SystemReserved:map[] HardEvictionThresholds:[]} QOSReserved:map[] ExperimentalCPUManagerPolicy:none ExperimentalTopologyManagerScope:container ExperimentalCPUManagerReconcilePeriod:10s ExperimentalMemoryManagerPolicy:None ExperimentalMemoryManagerReservedMemory:[] ExperimentalPodPidsLimit:-1 EnforceCPULimits:true CPUCFSQuotaPeriod:100ms ExperimentalTopologyManagerPolicy:none}
Aug 17 17:50:26 config-control-plane kubelet[286]: I0817 17:50:26.918355     286 topology_manager.go:120] "Creating topology manager with policy per scope" topologyPolicyName="none" topologyScopeName="container"
Aug 17 17:50:26 config-control-plane kubelet[286]: I0817 17:50:26.918375     286 container_manager_linux.go:314] "Initializing Topology Manager" policy="none" scope="container"
Aug 17 17:50:26 config-control-plane kubelet[286]: I0817 17:50:26.918398     286 container_manager_linux.go:319] "Creating device plugin manager" devicePluginEnabled=true
Aug 17 17:50:26 config-control-plane kubelet[286]: I0817 17:50:26.918616     286 remote_runtime.go:62] parsed scheme: ""
Aug 17 17:50:26 config-control-plane kubelet[286]: I0817 17:50:26.918640     286 remote_runtime.go:62] scheme "" not registered, fallback to default scheme
Aug 17 17:50:26 config-control-plane kubelet[286]: I0817 17:50:26.918673     286 passthrough.go:48] ccResolverWrapper: sending update to cc: {[{/run/containerd/containerd.sock  <nil> 0 <nil>}] <nil> <nil>}
Aug 17 17:50:26 config-control-plane kubelet[286]: I0817 17:50:26.918693     286 clientconn.go:948] ClientConn switching balancer to "pick_first"
Aug 17 17:50:26 config-control-plane kubelet[286]: I0817 17:50:26.918771     286 remote_image.go:50] parsed scheme: ""
Aug 17 17:50:26 config-control-plane kubelet[286]: I0817 17:50:26.918790     286 remote_image.go:50] scheme "" not registered, fallback to default scheme
Aug 17 17:50:26 config-control-plane kubelet[286]: I0817 17:50:26.918808     286 passthrough.go:48] ccResolverWrapper: sending update to cc: {[{/run/containerd/containerd.sock  <nil> 0 <nil>}] <nil> <nil>}
Aug 17 17:50:26 config-control-plane kubelet[286]: I0817 17:50:26.918821     286 clientconn.go:948] ClientConn switching balancer to "pick_first"
Aug 17 17:50:26 config-control-plane kubelet[286]: I0817 17:50:26.918888     286 kubelet.go:404] "Attempting to sync node with API server"
Aug 17 17:50:26 config-control-plane kubelet[286]: I0817 17:50:26.918923     286 kubelet.go:272] "Adding static pod path" path="/etc/kubernetes/manifests"
Aug 17 17:50:26 config-control-plane kubelet[286]: I0817 17:50:26.918958     286 kubelet.go:283] "Adding apiserver pod source"
Aug 17 17:50:26 config-control-plane kubelet[286]: I0817 17:50:26.918982     286 apiserver.go:42] "Waiting for node sync before watching apiserver pods"
Aug 17 17:50:26 config-control-plane kubelet[286]: E0817 17:50:26.920197     286 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.Service: failed to list *v1.Service: Get "https://config-control-plane:6443/api/v1/services?limit=500&resourceVersion=0": dial tcp 172.18.0.3:6443: connect: connection refused
Aug 17 17:50:26 config-control-plane kubelet[286]: E0817 17:50:26.920248     286 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.Node: failed to list *v1.Node: Get "https://config-control-plane:6443/api/v1/nodes?fieldSelector=metadata.name%3Dconfig-control-plane&limit=500&resourceVersion=0": dial tcp 172.18.0.3:6443: connect: connection refused
Aug 17 17:50:26 config-control-plane kubelet[286]: I0817 17:50:26.921971     286 kuberuntime_manager.go:222] "Container runtime initialized" containerRuntime="containerd" version="v1.5.1" apiVersion="v1alpha2"
Aug 17 17:50:27 config-control-plane kubelet[286]: E0817 17:50:27.808138     286 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.Node: failed to list *v1.Node: Get "https://config-control-plane:6443/api/v1/nodes?fieldSelector=metadata.name%3Dconfig-control-plane&limit=500&resourceVersion=0": dial tcp 172.18.0.3:6443: connect: connection refused
Aug 17 17:50:27 config-control-plane kubelet[286]: E0817 17:50:27.837652     286 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.Service: failed to list *v1.Service: Get "https://config-control-plane:6443/api/v1/services?limit=500&resourceVersion=0": dial tcp 172.18.0.3:6443: connect: connection refused
Aug 17 17:50:28 config-control-plane kubelet[286]: E0817 17:50:28.210267     286 certificate_manager.go:437] Failed while requesting a signed certificate from the master: cannot create certificate signing request: Post "https://config-control-plane:6443/apis/certificates.k8s.io/v1/certificatesigningrequests": dial tcp 172.18.0.3:6443: connect: connection refused
Aug 17 17:50:30 config-control-plane kubelet[286]: E0817 17:50:30.232862     286 aws_credentials.go:77] while getting AWS credentials NoCredentialProviders: no valid providers in chain. Deprecated.
Aug 17 17:50:30 config-control-plane kubelet[286]:         For verbose messaging see aws.Config.CredentialsChainVerboseErrors
Aug 17 17:50:30 config-control-plane kubelet[286]: W0817 17:50:30.233206     286 probe.go:268] Flexvolume plugin directory at /usr/libexec/kubernetes/kubelet-plugins/volume/exec/ does not exist. Recreating.
Aug 17 17:50:30 config-control-plane kubelet[286]: I0817 17:50:30.233725     286 server.go:1190] "Started kubelet"
Aug 17 17:50:30 config-control-plane kubelet[286]: I0817 17:50:30.234236     286 server.go:149] "Starting to listen" address="0.0.0.0" port=10250
Aug 17 17:50:30 config-control-plane kubelet[286]: E0817 17:50:30.235636     286 event.go:273] Unable to write event: '&v1.Event{TypeMeta:v1.TypeMeta{Kind:"", APIVersion:""}, ObjectMeta:v1.ObjectMeta{Name:"config-control-plane.169c2959a71b65d4", GenerateName:"", Namespace:"default", SelfLink:"", UID:"", ResourceVersion:"", Generation:0, CreationTimestamp:v1.Time{Time:time.Time{wall:0x0, ext:0, loc:(*time.Location)(nil)}}, DeletionTimestamp:(*v1.Time)(nil), DeletionGracePeriodSeconds:(*int64)(nil), Labels:map[string]string(nil), Annotations:map[string]string(nil), OwnerReferences:[]v1.OwnerReference(nil), Finalizers:[]string(nil), ClusterName:"", ManagedFields:[]v1.ManagedFieldsEntry(nil)}, InvolvedObject:v1.ObjectReference{Kind:"Node", Namespace:"", Name:"config-control-plane", UID:"config-control-plane", APIVersion:"", ResourceVersion:"", FieldPath:""}, Reason:"Starting", Message:"Starting kubelet.", Source:v1.EventSource{Component:"kubelet", Host:"config-control-plane"}, FirstTimestamp:v1.Time{Time:time.Time{wall:0xc03f1b998dede9d4, ext:8748535501, loc:(*time.Location)(0x74aba00)}}, LastTimestamp:v1.Time{Time:time.Time{wall:0xc03f1b998dede9d4, ext:8748535501, loc:(*time.Location)(0x74aba00)}}, Count:1, Type:"Normal", EventTime:v1.MicroTime{Time:time.Time{wall:0x0, ext:0, loc:(*time.Location)(nil)}}, Series:(*v1.EventSeries)(nil), Action:"", Related:(*v1.ObjectReference)(nil), ReportingController:"", ReportingInstance:""}': 'Post "https://config-control-plane:6443/api/v1/namespaces/default/events": dial tcp 172.18.0.3:6443: connect: connection refused'(may retry after sleeping)
Aug 17 17:50:30 config-control-plane kubelet[286]: I0817 17:50:30.235861     286 server.go:405] "Adding debug handlers to kubelet server"
Aug 17 17:50:30 config-control-plane kubelet[286]: I0817 17:50:30.236972     286 fs_resource_analyzer.go:67] "Starting FS ResourceAnalyzer"
Aug 17 17:50:30 config-control-plane kubelet[286]: I0817 17:50:30.237480     286 volume_manager.go:271] "Starting Kubelet Volume Manager"
Aug 17 17:50:30 config-control-plane kubelet[286]: I0817 17:50:30.237809     286 desired_state_of_world_populator.go:141] "Desired state populator starts to run"
Aug 17 17:50:30 config-control-plane kubelet[286]: E0817 17:50:30.238615     286 kubelet.go:2211] "Container runtime network not ready" networkReady="NetworkReady=false reason:NetworkPluginNotReady message:Network plugin returns error: cni plugin not initialized"
Aug 17 17:50:30 config-control-plane kubelet[286]: E0817 17:50:30.238826     286 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.CSIDriver: failed to list *v1.CSIDriver: Get "https://config-control-plane:6443/apis/storage.k8s.io/v1/csidrivers?limit=500&resourceVersion=0": dial tcp 172.18.0.3:6443: connect: connection refused
Aug 17 17:50:30 config-control-plane kubelet[286]: I0817 17:50:30.239313     286 client.go:86] parsed scheme: "unix"
Aug 17 17:50:30 config-control-plane kubelet[286]: I0817 17:50:30.239409     286 client.go:86] scheme "unix" not registered, fallback to default scheme
Aug 17 17:50:30 config-control-plane kubelet[286]: E0817 17:50:30.239004     286 controller.go:144] failed to ensure lease exists, will retry in 200ms, error: Get "https://config-control-plane:6443/apis/coordination.k8s.io/v1/namespaces/kube-node-lease/leases/config-control-plane?timeout=10s": dial tcp 172.18.0.3:6443: connect: connection refused
Aug 17 17:50:30 config-control-plane kubelet[286]: I0817 17:50:30.239635     286 passthrough.go:48] ccResolverWrapper: sending update to cc: {[{unix:///run/containerd/containerd.sock  <nil> 0 <nil>}] <nil> <nil>}
Aug 17 17:50:30 config-control-plane kubelet[286]: I0817 17:50:30.239673     286 clientconn.go:948] ClientConn switching balancer to "pick_first"
Aug 17 17:50:30 config-control-plane kubelet[286]: I0817 17:50:30.274455     286 cpu_manager.go:199] "Starting CPU manager" policy="none"
Aug 17 17:50:30 config-control-plane kubelet[286]: I0817 17:50:30.274497     286 cpu_manager.go:200] "Reconciling" reconcilePeriod="10s"
Aug 17 17:50:30 config-control-plane kubelet[286]: I0817 17:50:30.274522     286 state_mem.go:36] "Initialized new in-memory state store"
Aug 17 17:50:30 config-control-plane kubelet[286]: I0817 17:50:30.276290     286 policy_none.go:44] "None policy: Start"
Aug 17 17:50:30 config-control-plane kubelet[286]: I0817 17:50:30.288579     286 manager.go:600] "Failed to retrieve checkpoint" checkpoint="kubelet_internal_checkpoint" err="checkpoint is not found"
Aug 17 17:50:30 config-control-plane kubelet[286]: I0817 17:50:30.288801     286 plugin_manager.go:114] "Starting Kubelet Plugin Manager"
Aug 17 17:50:30 config-control-plane kubelet[286]: E0817 17:50:30.289960     286 eviction_manager.go:255] "Eviction manager: failed to get summary stats" err="failed to get node info: node \"config-control-plane\" not found"
Aug 17 17:50:30 config-control-plane kubelet[286]: E0817 17:50:30.338566     286 kubelet.go:2291] "Error getting node" err="node \"config-control-plane\" not found"
Aug 17 17:50:30 config-control-plane kubelet[286]: I0817 17:50:30.339666     286 kubelet_node_status.go:71] "Attempting to register node" node="config-control-plane"
Aug 17 17:50:30 config-control-plane kubelet[286]: E0817 17:50:30.340672     286 kubelet_node_status.go:93] "Unable to register node with API server" err="Post \"https://config-control-plane:6443/api/v1/nodes\": dial tcp 172.18.0.3:6443: connect: connection refused" node="config-control-plane"
Aug 17 17:50:30 config-control-plane kubelet[286]: E0817 17:50:30.438717     286 kubelet.go:2291] "Error getting node" err="node \"config-control-plane\" not found"
Aug 17 17:50:30 config-control-plane kubelet[286]: E0817 17:50:30.439302     286 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.Service: failed to list *v1.Service: Get "https://config-control-plane:6443/api/v1/services?limit=500&resourceVersion=0": dial tcp 172.18.0.3:6443: connect: connection refused
Aug 17 17:50:30 config-control-plane kubelet[286]: E0817 17:50:30.440115     286 controller.go:144] failed to ensure lease exists, will retry in 400ms, error: Get "https://config-control-plane:6443/apis/coordination.k8s.io/v1/namespaces/kube-node-lease/leases/config-control-plane?timeout=10s": dial tcp 172.18.0.3:6443: connect: connection refused
Aug 17 17:50:30 config-control-plane kubelet[286]: E0817 17:50:30.539376     286 kubelet.go:2291] "Error getting node" err="node \"config-control-plane\" not found"
Aug 17 17:50:30 config-control-plane kubelet[286]: I0817 17:50:30.541994     286 kubelet_node_status.go:71] "Attempting to register node" node="config-control-plane"
Aug 17 17:50:30 config-control-plane kubelet[286]: E0817 17:50:30.542883     286 kubelet_node_status.go:93] "Unable to register node with API server" err="Post \"https://config-control-plane:6443/api/v1/nodes\": dial tcp 172.18.0.3:6443: connect: connection refused" node="config-control-plane"
Aug 17 17:50:30 config-control-plane kubelet[286]: I0817 17:50:30.598432     286 kubelet_network_linux.go:56] "Initialized protocol iptables rules." protocol=IPv4
Aug 17 17:50:30 config-control-plane kubelet[286]: E0817 17:50:30.640217     286 kubelet.go:2291] "Error getting node" err="node \"config-control-plane\" not found"
Aug 17 17:50:30 config-control-plane kubelet[286]: E0817 17:50:30.740692     286 kubelet.go:2291] "Error getting node" err="node \"config-control-plane\" not found"
Aug 17 17:50:30 config-control-plane kubelet[286]: E0817 17:50:30.750119     286 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.Node: failed to list *v1.Node: Get "https://config-control-plane:6443/api/v1/nodes?fieldSelector=metadata.name%3Dconfig-control-plane&limit=500&resourceVersion=0": dial tcp 172.18.0.3:6443: connect: connection refused
Aug 17 17:50:30 config-control-plane kubelet[286]: E0817 17:50:30.840830     286 kubelet.go:2291] "Error getting node" err="node \"config-control-plane\" not found"
Aug 17 17:50:30 config-control-plane kubelet[286]: E0817 17:50:30.841379     286 controller.go:144] failed to ensure lease exists, will retry in 800ms, error: Get "https://config-control-plane:6443/apis/coordination.k8s.io/v1/namespaces/kube-node-lease/leases/config-control-plane?timeout=10s": dial tcp 172.18.0.3:6443: connect: connection refused
Aug 17 17:50:30 config-control-plane kubelet[286]: E0817 17:50:30.942159     286 kubelet.go:2291] "Error getting node" err="node \"config-control-plane\" not found"
Aug 17 17:50:30 config-control-plane kubelet[286]: I0817 17:50:30.944976     286 kubelet_node_status.go:71] "Attempting to register node" node="config-control-plane"
Aug 17 17:50:30 config-control-plane kubelet[286]: E0817 17:50:30.946087     286 kubelet_node_status.go:93] "Unable to register node with API server" err="Post \"https://config-control-plane:6443/api/v1/nodes\": dial tcp 172.18.0.3:6443: connect: connection refused" node="config-control-plane"
Aug 17 17:50:30 config-control-plane kubelet[286]: I0817 17:50:30.961453     286 kubelet_network_linux.go:56] "Initialized protocol iptables rules." protocol=IPv6
Aug 17 17:50:30 config-control-plane kubelet[286]: I0817 17:50:30.961501     286 status_manager.go:157] "Starting to sync pod status with apiserver"
Aug 17 17:50:30 config-control-plane kubelet[286]: I0817 17:50:30.961533     286 kubelet.go:1846] "Starting kubelet main sync loop"
Aug 17 17:50:30 config-control-plane kubelet[286]: E0817 17:50:30.961595     286 kubelet.go:1870] "Skipping pod synchronization" err="PLEG is not healthy: pleg has yet to be successful"
Aug 17 17:50:30 config-control-plane kubelet[286]: E0817 17:50:30.962915     286 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.RuntimeClass: failed to list *v1.RuntimeClass: Get "https://config-control-plane:6443/apis/node.k8s.io/v1/runtimeclasses?limit=500&resourceVersion=0": dial tcp 172.18.0.3:6443: connect: connection refused
Aug 17 17:50:31 config-control-plane kubelet[286]: E0817 17:50:31.042763     286 kubelet.go:2291] "Error getting node" err="node \"config-control-plane\" not found"
Aug 17 17:50:31 config-control-plane kubelet[286]: I0817 17:50:31.062099     286 topology_manager.go:187] "Topology Admit Handler"
Aug 17 17:50:31 config-control-plane kubelet[286]: I0817 17:50:31.063385     286 topology_manager.go:187] "Topology Admit Handler"
Aug 17 17:50:31 config-control-plane kubelet[286]: I0817 17:50:31.064402     286 topology_manager.go:187] "Topology Admit Handler"
Aug 17 17:50:31 config-control-plane kubelet[286]: I0817 17:50:31.065586     286 topology_manager.go:187] "Topology Admit Handler"
Aug 17 17:50:31 config-control-plane kubelet[286]: I0817 17:50:31.067121     286 status_manager.go:566] "Failed to get status for pod" podUID=72c47c236abf4280d9f8b6af5ec1850d pod="kube-system/etcd-config-control-plane" error="Get \"https://config-control-plane:6443/api/v1/namespaces/kube-system/pods/etcd-config-control-plane\": dial tcp 172.18.0.3:6443: connect: connection refused"
Aug 17 17:50:31 config-control-plane kubelet[286]: I0817 17:50:31.068380     286 status_manager.go:566] "Failed to get status for pod" podUID=6f9f8d44cbdd820f10746e0421ee018f pod="kube-system/kube-apiserver-config-control-plane" error="Get \"https://config-control-plane:6443/api/v1/namespaces/kube-system/pods/kube-apiserver-config-control-plane\": dial tcp 172.18.0.3:6443: connect: connection refused"
Aug 17 17:50:31 config-control-plane kubelet[286]: I0817 17:50:31.069014     286 status_manager.go:566] "Failed to get status for pod" podUID=bd0bc82861de35a27312a75b3201e56c pod="kube-system/kube-controller-manager-config-control-plane" error="Get \"https://config-control-plane:6443/api/v1/namespaces/kube-system/pods/kube-controller-manager-config-control-plane\": dial tcp 172.18.0.3:6443: connect: connection refused"
Aug 17 17:50:31 config-control-plane kubelet[286]: I0817 17:50:31.069954     286 status_manager.go:566] "Failed to get status for pod" podUID=18ab771e4cf3f5b60acc165ef57a754d pod="kube-system/kube-scheduler-config-control-plane" error="Get \"https://config-control-plane:6443/api/v1/namespaces/kube-system/pods/kube-scheduler-config-control-plane\": dial tcp 172.18.0.3:6443: connect: connection refused"
Aug 17 17:50:31 config-control-plane kubelet[286]: I0817 17:50:31.142618     286 reconciler.go:224] "operationExecutor.VerifyControllerAttachedVolume started for volume \"etcd-data\" (UniqueName: \"kubernetes.io/host-path/72c47c236abf4280d9f8b6af5ec1850d-etcd-data\") pod \"etcd-config-control-plane\" (UID: \"72c47c236abf4280d9f8b6af5ec1850d\") "
Aug 17 17:50:31 config-control-plane kubelet[286]: I0817 17:50:31.142680     286 reconciler.go:224] "operationExecutor.VerifyControllerAttachedVolume started for volume \"ca-certs\" (UniqueName: \"kubernetes.io/host-path/bd0bc82861de35a27312a75b3201e56c-ca-certs\") pod \"kube-controller-manager-config-control-plane\" (UID: \"bd0bc82861de35a27312a75b3201e56c\") "
Aug 17 17:50:31 config-control-plane kubelet[286]: I0817 17:50:31.142723     286 reconciler.go:224] "operationExecutor.VerifyControllerAttachedVolume started for volume \"k8s-certs\" (UniqueName: \"kubernetes.io/host-path/bd0bc82861de35a27312a75b3201e56c-k8s-certs\") pod \"kube-controller-manager-config-control-plane\" (UID: \"bd0bc82861de35a27312a75b3201e56c\") "
Aug 17 17:50:31 config-control-plane kubelet[286]: I0817 17:50:31.142763     286 reconciler.go:224] "operationExecutor.VerifyControllerAttachedVolume started for volume \"usr-local-share-ca-certificates\" (UniqueName: \"kubernetes.io/host-path/bd0bc82861de35a27312a75b3201e56c-usr-local-share-ca-certificates\") pod \"kube-controller-manager-config-control-plane\" (UID: \"bd0bc82861de35a27312a75b3201e56c\") "
Aug 17 17:50:31 config-control-plane kubelet[286]: I0817 17:50:31.142822     286 reconciler.go:224] "operationExecutor.VerifyControllerAttachedVolume started for volume \"usr-share-ca-certificates\" (UniqueName: \"kubernetes.io/host-path/bd0bc82861de35a27312a75b3201e56c-usr-share-ca-certificates\") pod \"kube-controller-manager-config-control-plane\" (UID: \"bd0bc82861de35a27312a75b3201e56c\") "
Aug 17 17:50:31 config-control-plane kubelet[286]: I0817 17:50:31.142864     286 reconciler.go:224] "operationExecutor.VerifyControllerAttachedVolume started for volume \"usr-local-share-ca-certificates\" (UniqueName: \"kubernetes.io/host-path/6f9f8d44cbdd820f10746e0421ee018f-usr-local-share-ca-certificates\") pod \"kube-apiserver-config-control-plane\" (UID: \"6f9f8d44cbdd820f10746e0421ee018f\") "
Aug 17 17:50:31 config-control-plane kubelet[286]: I0817 17:50:31.142901     286 reconciler.go:224] "operationExecutor.VerifyControllerAttachedVolume started for volume \"etc-ca-certificates\" (UniqueName: \"kubernetes.io/host-path/bd0bc82861de35a27312a75b3201e56c-etc-ca-certificates\") pod \"kube-controller-manager-config-control-plane\" (UID: \"bd0bc82861de35a27312a75b3201e56c\") "
Aug 17 17:50:31 config-control-plane kubelet[286]: I0817 17:50:31.142937     286 reconciler.go:224] "operationExecutor.VerifyControllerAttachedVolume started for volume \"kubeconfig\" (UniqueName: \"kubernetes.io/host-path/18ab771e4cf3f5b60acc165ef57a754d-kubeconfig\") pod \"kube-scheduler-config-control-plane\" (UID: \"18ab771e4cf3f5b60acc165ef57a754d\") "
Aug 17 17:50:31 config-control-plane kubelet[286]: I0817 17:50:31.142973     286 reconciler.go:224] "operationExecutor.VerifyControllerAttachedVolume started for volume \"etcd-certs\" (UniqueName: \"kubernetes.io/host-path/72c47c236abf4280d9f8b6af5ec1850d-etcd-certs\") pod \"etcd-config-control-plane\" (UID: \"72c47c236abf4280d9f8b6af5ec1850d\") "
Aug 17 17:50:31 config-control-plane kubelet[286]: I0817 17:50:31.143009     286 reconciler.go:224] "operationExecutor.VerifyControllerAttachedVolume started for volume \"k8s-certs\" (UniqueName: \"kubernetes.io/host-path/6f9f8d44cbdd820f10746e0421ee018f-k8s-certs\") pod \"kube-apiserver-config-control-plane\" (UID: \"6f9f8d44cbdd820f10746e0421ee018f\") "
Aug 17 17:50:31 config-control-plane kubelet[286]: I0817 17:50:31.143087     286 reconciler.go:224] "operationExecutor.VerifyControllerAttachedVolume started for volume \"usr-share-ca-certificates\" (UniqueName: \"kubernetes.io/host-path/6f9f8d44cbdd820f10746e0421ee018f-usr-share-ca-certificates\") pod \"kube-apiserver-config-control-plane\" (UID: \"6f9f8d44cbdd820f10746e0421ee018f\") "
Aug 17 17:50:31 config-control-plane kubelet[286]: I0817 17:50:31.143123     286 reconciler.go:224] "operationExecutor.VerifyControllerAttachedVolume started for volume \"flexvolume-dir\" (UniqueName: \"kubernetes.io/host-path/bd0bc82861de35a27312a75b3201e56c-flexvolume-dir\") pod \"kube-controller-manager-config-control-plane\" (UID: \"bd0bc82861de35a27312a75b3201e56c\") "
Aug 17 17:50:31 config-control-plane kubelet[286]: I0817 17:50:31.143154     286 reconciler.go:224] "operationExecutor.VerifyControllerAttachedVolume started for volume \"kubeconfig\" (UniqueName: \"kubernetes.io/host-path/bd0bc82861de35a27312a75b3201e56c-kubeconfig\") pod \"kube-controller-manager-config-control-plane\" (UID: \"bd0bc82861de35a27312a75b3201e56c\") "
Aug 17 17:50:31 config-control-plane kubelet[286]: I0817 17:50:31.143193     286 reconciler.go:224] "operationExecutor.VerifyControllerAttachedVolume started for volume \"ca-certs\" (UniqueName: \"kubernetes.io/host-path/6f9f8d44cbdd820f10746e0421ee018f-ca-certs\") pod \"kube-apiserver-config-control-plane\" (UID: \"6f9f8d44cbdd820f10746e0421ee018f\") "
Aug 17 17:50:31 config-control-plane kubelet[286]: I0817 17:50:31.143233     286 reconciler.go:224] "operationExecutor.VerifyControllerAttachedVolume started for volume \"etc-ca-certificates\" (UniqueName: \"kubernetes.io/host-path/6f9f8d44cbdd820f10746e0421ee018f-etc-ca-certificates\") pod \"kube-apiserver-config-control-plane\" (UID: \"6f9f8d44cbdd820f10746e0421ee018f\") "
Aug 17 17:50:31 config-control-plane kubelet[286]: E0817 17:50:31.143301     286 kubelet.go:2291] "Error getting node" err="node \"config-control-plane\" not found"
Aug 17 17:50:31 config-control-plane kubelet[286]: E0817 17:50:31.170475     286 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.CSIDriver: failed to list *v1.CSIDriver: Get "https://config-control-plane:6443/apis/storage.k8s.io/v1/csidrivers?limit=500&resourceVersion=0": dial tcp 172.18.0.3:6443: connect: connection refused
Aug 17 17:50:31 config-control-plane kubelet[286]: E0817 17:50:31.244472     286 kubelet.go:2291] "Error getting node" err="node \"config-control-plane\" not found"
Aug 17 17:50:31 config-control-plane kubelet[286]: E0817 17:50:31.344948     286 kubelet.go:2291] "Error getting node" err="node \"config-control-plane\" not found"
Aug 17 17:50:31 config-control-plane kubelet[286]: E0817 17:50:31.446035     286 kubelet.go:2291] "Error getting node" err="node \"config-control-plane\" not found"
Aug 17 17:50:31 config-control-plane kubelet[286]: E0817 17:50:31.546973     286 kubelet.go:2291] "Error getting node" err="node \"config-control-plane\" not found"
Aug 17 17:50:31 config-control-plane kubelet[286]: E0817 17:50:31.642360     286 controller.go:144] failed to ensure lease exists, will retry in 1.6s, error: Get "https://config-control-plane:6443/apis/coordination.k8s.io/v1/namespaces/kube-node-lease/leases/config-control-plane?timeout=10s": dial tcp 172.18.0.3:6443: connect: connection refused
Aug 17 17:50:31 config-control-plane kubelet[286]: E0817 17:50:31.647725     286 kubelet.go:2291] "Error getting node" err="node \"config-control-plane\" not found"
Aug 17 17:50:31 config-control-plane kubelet[286]: E0817 17:50:31.748215     286 kubelet.go:2291] "Error getting node" err="node \"config-control-plane\" not found"
Aug 17 17:50:31 config-control-plane kubelet[286]: I0817 17:50:31.749407     286 kubelet_node_status.go:71] "Attempting to register node" node="config-control-plane"
Aug 17 17:50:31 config-control-plane kubelet[286]: E0817 17:50:31.751279     286 kubelet_node_status.go:93] "Unable to register node with API server" err="Post \"https://config-control-plane:6443/api/v1/nodes\": dial tcp 172.18.0.3:6443: connect: connection refused" node="config-control-plane"
Aug 17 17:50:31 config-control-plane kubelet[286]: E0817 17:50:31.849378     286 kubelet.go:2291] "Error getting node" err="node \"config-control-plane\" not found"
Aug 17 17:50:31 config-control-plane kubelet[286]: E0817 17:50:31.949780     286 kubelet.go:2291] "Error getting node" err="node \"config-control-plane\" not found"
Aug 17 17:50:31 config-control-plane kubelet[286]: E0817 17:50:31.977599     286 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.RuntimeClass: failed to list *v1.RuntimeClass: Get "https://config-control-plane:6443/apis/node.k8s.io/v1/runtimeclasses?limit=500&resourceVersion=0": dial tcp 172.18.0.3:6443: connect: connection refused
Aug 17 17:50:32 config-control-plane kubelet[286]: E0817 17:50:32.050887     286 kubelet.go:2291] "Error getting node" err="node \"config-control-plane\" not found"
Aug 17 17:50:32 config-control-plane kubelet[286]: E0817 17:50:32.152103     286 kubelet.go:2291] "Error getting node" err="node \"config-control-plane\" not found"
Aug 17 17:50:32 config-control-plane kubelet[286]: E0817 17:50:32.252998     286 kubelet.go:2291] "Error getting node" err="node \"config-control-plane\" not found"
Aug 17 17:50:32 config-control-plane kubelet[286]: E0817 17:50:32.353893     286 kubelet.go:2291] "Error getting node" err="node \"config-control-plane\" not found"
Aug 17 17:50:32 config-control-plane kubelet[286]: E0817 17:50:32.454580     286 kubelet.go:2291] "Error getting node" err="node \"config-control-plane\" not found"
Aug 17 17:50:32 config-control-plane kubelet[286]: E0817 17:50:32.554765     286 kubelet.go:2291] "Error getting node" err="node \"config-control-plane\" not found"
Aug 17 17:50:32 config-control-plane kubelet[286]: E0817 17:50:32.654939     286 kubelet.go:2291] "Error getting node" err="node \"config-control-plane\" not found"
Aug 17 17:50:32 config-control-plane kubelet[286]: E0817 17:50:32.755732     286 kubelet.go:2291] "Error getting node" err="node \"config-control-plane\" not found"
Aug 17 17:50:32 config-control-plane kubelet[286]: E0817 17:50:32.856365     286 kubelet.go:2291] "Error getting node" err="node \"config-control-plane\" not found"
Aug 17 17:50:32 config-control-plane kubelet[286]: E0817 17:50:32.956891     286 kubelet.go:2291] "Error getting node" err="node \"config-control-plane\" not found"
Aug 17 17:50:33 config-control-plane kubelet[286]: E0817 17:50:33.058121     286 kubelet.go:2291] "Error getting node" err="node \"config-control-plane\" not found"
Aug 17 17:50:33 config-control-plane kubelet[286]: E0817 17:50:33.158835     286 kubelet.go:2291] "Error getting node" err="node \"config-control-plane\" not found"
Aug 17 17:50:33 config-control-plane kubelet[286]: E0817 17:50:33.244266     286 controller.go:144] failed to ensure lease exists, will retry in 3.2s, error: Get "https://config-control-plane:6443/apis/coordination.k8s.io/v1/namespaces/kube-node-lease/leases/config-control-plane?timeout=10s": dial tcp 172.18.0.3:6443: connect: connection refused
Aug 17 17:50:33 config-control-plane kubelet[286]: E0817 17:50:33.259561     286 kubelet.go:2291] "Error getting node" err="node \"config-control-plane\" not found"
Aug 17 17:50:33 config-control-plane kubelet[286]: I0817 17:50:33.353194     286 kubelet_node_status.go:71] "Attempting to register node" node="config-control-plane"
Aug 17 17:50:33 config-control-plane kubelet[286]: E0817 17:50:33.354256     286 kubelet_node_status.go:93] "Unable to register node with API server" err="Post \"https://config-control-plane:6443/api/v1/nodes\": dial tcp 172.18.0.3:6443: connect: connection refused" node="config-control-plane"
Aug 17 17:50:33 config-control-plane kubelet[286]: E0817 17:50:33.360481     286 kubelet.go:2291] "Error getting node" err="node \"config-control-plane\" not found"
Aug 17 17:50:33 config-control-plane kubelet[286]: E0817 17:50:33.441004     286 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.CSIDriver: failed to list *v1.CSIDriver: Get "https://config-control-plane:6443/apis/storage.k8s.io/v1/csidrivers?limit=500&resourceVersion=0": dial tcp 172.18.0.3:6443: connect: connection refused
Aug 17 17:50:33 config-control-plane kubelet[286]: E0817 17:50:33.461590     286 kubelet.go:2291] "Error getting node" err="node \"config-control-plane\" not found"
Aug 17 17:50:33 config-control-plane kubelet[286]: E0817 17:50:33.561875     286 kubelet.go:2291] "Error getting node" err="node \"config-control-plane\" not found"
Aug 17 17:50:33 config-control-plane kubelet[286]: E0817 17:50:33.665184     286 kubelet.go:2291] "Error getting node" err="node \"config-control-plane\" not found"
Aug 17 17:50:33 config-control-plane kubelet[286]: E0817 17:50:33.768246     286 kubelet.go:2291] "Error getting node" err="node \"config-control-plane\" not found"
Aug 17 17:50:33 config-control-plane kubelet[286]: E0817 17:50:33.868791     286 kubelet.go:2291] "Error getting node" err="node \"config-control-plane\" not found"
Aug 17 17:50:33 config-control-plane kubelet[286]: E0817 17:50:33.969872     286 kubelet.go:2291] "Error getting node" err="node \"config-control-plane\" not found"
Aug 17 17:50:34 config-control-plane kubelet[286]: I0817 17:50:34.020361     286 status_manager.go:566] "Failed to get status for pod" podUID=bd0bc82861de35a27312a75b3201e56c pod="kube-system/kube-controller-manager-config-control-plane" error="Get \"https://config-control-plane:6443/api/v1/namespaces/kube-system/pods/kube-controller-manager-config-control-plane\": dial tcp 172.18.0.3:6443: connect: connection refused"
Aug 17 17:50:34 config-control-plane kubelet[286]: I0817 17:50:34.031569     286 status_manager.go:566] "Failed to get status for pod" podUID=6f9f8d44cbdd820f10746e0421ee018f pod="kube-system/kube-apiserver-config-control-plane" error="Get \"https://config-control-plane:6443/api/v1/namespaces/kube-system/pods/kube-apiserver-config-control-plane\": dial tcp 172.18.0.3:6443: connect: connection refused"
Aug 17 17:50:34 config-control-plane kubelet[286]: I0817 17:50:34.042006     286 status_manager.go:566] "Failed to get status for pod" podUID=18ab771e4cf3f5b60acc165ef57a754d pod="kube-system/kube-scheduler-config-control-plane" error="Get \"https://config-control-plane:6443/api/v1/namespaces/kube-system/pods/kube-scheduler-config-control-plane\": dial tcp 172.18.0.3:6443: connect: connection refused"
Aug 17 17:50:34 config-control-plane kubelet[286]: E0817 17:50:34.071175     286 kubelet.go:2291] "Error getting node" err="node \"config-control-plane\" not found"
Aug 17 17:50:34 config-control-plane kubelet[286]: E0817 17:50:34.172434     286 kubelet.go:2291] "Error getting node" err="node \"config-control-plane\" not found"
Aug 17 17:50:34 config-control-plane kubelet[286]: E0817 17:50:34.273510     286 kubelet.go:2291] "Error getting node" err="node \"config-control-plane\" not found"
Aug 17 17:50:34 config-control-plane kubelet[286]: E0817 17:50:34.383200     286 kubelet.go:2291] "Error getting node" err="node \"config-control-plane\" not found"
Aug 17 17:50:34 config-control-plane kubelet[286]: E0817 17:50:34.484149     286 kubelet.go:2291] "Error getting node" err="node \"config-control-plane\" not found"
Aug 17 17:50:34 config-control-plane kubelet[286]: E0817 17:50:34.584313     286 kubelet.go:2291] "Error getting node" err="node \"config-control-plane\" not found"
Aug 17 17:50:34 config-control-plane kubelet[286]: E0817 17:50:34.685109     286 kubelet.go:2291] "Error getting node" err="node \"config-control-plane\" not found"
Aug 17 17:50:34 config-control-plane kubelet[286]: E0817 17:50:34.785685     286 kubelet.go:2291] "Error getting node" err="node \"config-control-plane\" not found"
Aug 17 17:50:34 config-control-plane kubelet[286]: E0817 17:50:34.886510     286 kubelet.go:2291] "Error getting node" err="node \"config-control-plane\" not found"
Aug 17 17:50:34 config-control-plane kubelet[286]: E0817 17:50:34.986799     286 kubelet.go:2291] "Error getting node" err="node \"config-control-plane\" not found"
Aug 17 17:50:35 config-control-plane kubelet[286]: E0817 17:50:35.087064     286 kubelet.go:2291] "Error getting node" err="node \"config-control-plane\" not found"
Aug 17 17:50:35 config-control-plane kubelet[286]: E0817 17:50:35.188697     286 kubelet.go:2291] "Error getting node" err="node \"config-control-plane\" not found"
Aug 17 17:50:35 config-control-plane kubelet[286]: E0817 17:50:35.292644     286 kubelet.go:2291] "Error getting node" err="node \"config-control-plane\" not found"
Aug 17 17:50:35 config-control-plane kubelet[286]: E0817 17:50:35.294579     286 kubelet.go:2211] "Container runtime network not ready" networkReady="NetworkReady=false reason:NetworkPluginNotReady message:Network plugin returns error: cni plugin not initialized"
Aug 17 17:50:35 config-control-plane kubelet[286]: E0817 17:50:35.393400     286 kubelet.go:2291] "Error getting node" err="node \"config-control-plane\" not found"
Aug 17 17:50:35 config-control-plane kubelet[286]: E0817 17:50:35.494360     286 kubelet.go:2291] "Error getting node" err="node \"config-control-plane\" not found"
Aug 17 17:50:35 config-control-plane kubelet[286]: E0817 17:50:35.595624     286 kubelet.go:2291] "Error getting node" err="node \"config-control-plane\" not found"
Aug 17 17:50:35 config-control-plane kubelet[286]: E0817 17:50:35.695981     286 kubelet.go:2291] "Error getting node" err="node \"config-control-plane\" not found"
Aug 17 17:50:35 config-control-plane kubelet[286]: E0817 17:50:35.796118     286 kubelet.go:2291] "Error getting node" err="node \"config-control-plane\" not found"
Aug 17 17:50:35 config-control-plane kubelet[286]: E0817 17:50:35.897340     286 kubelet.go:2291] "Error getting node" err="node \"config-control-plane\" not found"
Aug 17 17:50:35 config-control-plane kubelet[286]: E0817 17:50:35.997826     286 kubelet.go:2291] "Error getting node" err="node \"config-control-plane\" not found"
Aug 17 17:50:36 config-control-plane kubelet[286]: E0817 17:50:36.098952     286 kubelet.go:2291] "Error getting node" err="node \"config-control-plane\" not found"
Aug 17 17:50:36 config-control-plane kubelet[286]: E0817 17:50:36.199573     286 kubelet.go:2291] "Error getting node" err="node \"config-control-plane\" not found"
Aug 17 17:50:36 config-control-plane kubelet[286]: E0817 17:50:36.300982     286 kubelet.go:2291] "Error getting node" err="node \"config-control-plane\" not found"
Aug 17 17:50:36 config-control-plane kubelet[286]: E0817 17:50:36.401743     286 kubelet.go:2291] "Error getting node" err="node \"config-control-plane\" not found"
Aug 17 17:50:36 config-control-plane kubelet[286]: E0817 17:50:36.502801     286 kubelet.go:2291] "Error getting node" err="node \"config-control-plane\" not found"
Aug 17 17:50:36 config-control-plane kubelet[286]: I0817 17:50:36.556652     286 kubelet_node_status.go:71] "Attempting to register node" node="config-control-plane"
Aug 17 17:50:36 config-control-plane kubelet[286]: E0817 17:50:36.603267     286 kubelet.go:2291] "Error getting node" err="node \"config-control-plane\" not found"
Aug 17 17:50:36 config-control-plane kubelet[286]: E0817 17:50:36.704466     286 kubelet.go:2291] "Error getting node" err="node \"config-control-plane\" not found"
Aug 17 17:50:36 config-control-plane kubelet[286]: E0817 17:50:36.805194     286 kubelet.go:2291] "Error getting node" err="node \"config-control-plane\" not found"
Aug 17 17:50:36 config-control-plane kubelet[286]: E0817 17:50:36.906371     286 kubelet.go:2291] "Error getting node" err="node \"config-control-plane\" not found"
Aug 17 17:50:37 config-control-plane kubelet[286]: E0817 17:50:37.007183     286 kubelet.go:2291] "Error getting node" err="node \"config-control-plane\" not found"
Aug 17 17:50:37 config-control-plane kubelet[286]: E0817 17:50:37.107890     286 kubelet.go:2291] "Error getting node" err="node \"config-control-plane\" not found"
Aug 17 17:50:37 config-control-plane kubelet[286]: E0817 17:50:37.208003     286 kubelet.go:2291] "Error getting node" err="node \"config-control-plane\" not found"
Aug 17 17:50:37 config-control-plane kubelet[286]: E0817 17:50:37.308887     286 kubelet.go:2291] "Error getting node" err="node \"config-control-plane\" not found"
Aug 17 17:50:37 config-control-plane kubelet[286]: E0817 17:50:37.409782     286 kubelet.go:2291] "Error getting node" err="node \"config-control-plane\" not found"
Aug 17 17:50:37 config-control-plane kubelet[286]: E0817 17:50:37.510281     286 kubelet.go:2291] "Error getting node" err="node \"config-control-plane\" not found"
Aug 17 17:50:37 config-control-plane kubelet[286]: E0817 17:50:37.611258     286 kubelet.go:2291] "Error getting node" err="node \"config-control-plane\" not found"
Aug 17 17:50:37 config-control-plane kubelet[286]: E0817 17:50:37.712182     286 kubelet.go:2291] "Error getting node" err="node \"config-control-plane\" not found"
Aug 17 17:50:37 config-control-plane kubelet[286]: E0817 17:50:37.812411     286 kubelet.go:2291] "Error getting node" err="node \"config-control-plane\" not found"
Aug 17 17:50:37 config-control-plane kubelet[286]: E0817 17:50:37.913429     286 kubelet.go:2291] "Error getting node" err="node \"config-control-plane\" not found"
Aug 17 17:50:38 config-control-plane kubelet[286]: E0817 17:50:38.014099     286 kubelet.go:2291] "Error getting node" err="node \"config-control-plane\" not found"
Aug 17 17:50:38 config-control-plane kubelet[286]: E0817 17:50:38.114783     286 kubelet.go:2291] "Error getting node" err="node \"config-control-plane\" not found"
Aug 17 17:50:38 config-control-plane kubelet[286]: E0817 17:50:38.215292     286 kubelet.go:2291] "Error getting node" err="node \"config-control-plane\" not found"
Aug 17 17:50:38 config-control-plane kubelet[286]: E0817 17:50:38.315971     286 kubelet.go:2291] "Error getting node" err="node \"config-control-plane\" not found"
Aug 17 17:50:38 config-control-plane kubelet[286]: E0817 17:50:38.416297     286 kubelet.go:2291] "Error getting node" err="node \"config-control-plane\" not found"
Aug 17 17:50:38 config-control-plane kubelet[286]: E0817 17:50:38.517576     286 kubelet.go:2291] "Error getting node" err="node \"config-control-plane\" not found"
Aug 17 17:50:38 config-control-plane kubelet[286]: E0817 17:50:38.618135     286 kubelet.go:2291] "Error getting node" err="node \"config-control-plane\" not found"
Aug 17 17:50:38 config-control-plane kubelet[286]: E0817 17:50:38.718277     286 kubelet.go:2291] "Error getting node" err="node \"config-control-plane\" not found"
Aug 17 17:50:38 config-control-plane kubelet[286]: E0817 17:50:38.819219     286 kubelet.go:2291] "Error getting node" err="node \"config-control-plane\" not found"
Aug 17 17:50:38 config-control-plane kubelet[286]: E0817 17:50:38.919998     286 kubelet.go:2291] "Error getting node" err="node \"config-control-plane\" not found"
Aug 17 17:50:39 config-control-plane kubelet[286]: E0817 17:50:39.021031     286 kubelet.go:2291] "Error getting node" err="node \"config-control-plane\" not found"
Aug 17 17:50:39 config-control-plane kubelet[286]: E0817 17:50:39.122083     286 kubelet.go:2291] "Error getting node" err="node \"config-control-plane\" not found"
Aug 17 17:50:39 config-control-plane kubelet[286]: E0817 17:50:39.223117     286 kubelet.go:2291] "Error getting node" err="node \"config-control-plane\" not found"
Aug 17 17:50:39 config-control-plane kubelet[286]: E0817 17:50:39.323318     286 kubelet.go:2291] "Error getting node" err="node \"config-control-plane\" not found"
Aug 17 17:50:39 config-control-plane kubelet[286]: E0817 17:50:39.424176     286 kubelet.go:2291] "Error getting node" err="node \"config-control-plane\" not found"
Aug 17 17:50:39 config-control-plane kubelet[286]: E0817 17:50:39.524276     286 kubelet.go:2291] "Error getting node" err="node \"config-control-plane\" not found"
Aug 17 17:50:39 config-control-plane kubelet[286]: E0817 17:50:39.625261     286 kubelet.go:2291] "Error getting node" err="node \"config-control-plane\" not found"
Aug 17 17:50:39 config-control-plane kubelet[286]: E0817 17:50:39.725427     286 kubelet.go:2291] "Error getting node" err="node \"config-control-plane\" not found"
Aug 17 17:50:39 config-control-plane kubelet[286]: E0817 17:50:39.826576     286 kubelet.go:2291] "Error getting node" err="node \"config-control-plane\" not found"
Aug 17 17:50:39 config-control-plane kubelet[286]: E0817 17:50:39.927730     286 kubelet.go:2291] "Error getting node" err="node \"config-control-plane\" not found"
Aug 17 17:50:40 config-control-plane kubelet[286]: E0817 17:50:40.028275     286 kubelet.go:2291] "Error getting node" err="node \"config-control-plane\" not found"
Aug 17 17:50:40 config-control-plane kubelet[286]: E0817 17:50:40.128885     286 kubelet.go:2291] "Error getting node" err="node \"config-control-plane\" not found"
Aug 17 17:50:40 config-control-plane kubelet[286]: E0817 17:50:40.229626     286 kubelet.go:2291] "Error getting node" err="node \"config-control-plane\" not found"
Aug 17 17:50:40 config-control-plane kubelet[286]: E0817 17:50:40.290727     286 eviction_manager.go:255] "Eviction manager: failed to get summary stats" err="failed to get node info: node \"config-control-plane\" not found"
Aug 17 17:50:40 config-control-plane kubelet[286]: E0817 17:50:40.297772     286 kubelet.go:2211] "Container runtime network not ready" networkReady="NetworkReady=false reason:NetworkPluginNotReady message:Network plugin returns error: cni plugin not initialized"
Aug 17 17:50:40 config-control-plane kubelet[286]: E0817 17:50:40.331008     286 kubelet.go:2291] "Error getting node" err="node \"config-control-plane\" not found"
Aug 17 17:50:40 config-control-plane kubelet[286]: E0817 17:50:40.432349     286 kubelet.go:2291] "Error getting node" err="node \"config-control-plane\" not found"
Aug 17 17:50:40 config-control-plane kubelet[286]: E0817 17:50:40.533567     286 kubelet.go:2291] "Error getting node" err="node \"config-control-plane\" not found"
Aug 17 17:50:40 config-control-plane kubelet[286]: E0817 17:50:40.636452     286 kubelet.go:2291] "Error getting node" err="node \"config-control-plane\" not found"
Aug 17 17:50:40 config-control-plane kubelet[286]: E0817 17:50:40.740403     286 kubelet.go:2291] "Error getting node" err="node \"config-control-plane\" not found"
Aug 17 17:50:40 config-control-plane kubelet[286]: E0817 17:50:40.772908     286 nodelease.go:49] "Failed to get node when trying to set owner ref to the node lease" err="nodes \"config-control-plane\" not found" node="config-control-plane"
Aug 17 17:50:40 config-control-plane kubelet[286]: E0817 17:50:40.841213     286 kubelet.go:2291] "Error getting node" err="node \"config-control-plane\" not found"
Aug 17 17:50:40 config-control-plane kubelet[286]: I0817 17:50:40.860403     286 kubelet_node_status.go:74] "Successfully registered node" node="config-control-plane"
Aug 17 17:50:40 config-control-plane kubelet[286]: E0817 17:50:40.901271     286 event.go:264] Server rejected event '&v1.Event{TypeMeta:v1.TypeMeta{Kind:"", APIVersion:""}, ObjectMeta:v1.ObjectMeta{Name:"config-control-plane.169c2959a71b65d4", GenerateName:"", Namespace:"default", SelfLink:"", UID:"", ResourceVersion:"", Generation:0, CreationTimestamp:v1.Time{Time:time.Time{wall:0x0, ext:0, loc:(*time.Location)(nil)}}, DeletionTimestamp:(*v1.Time)(nil), DeletionGracePeriodSeconds:(*int64)(nil), Labels:map[string]string(nil), Annotations:map[string]string(nil), OwnerReferences:[]v1.OwnerReference(nil), Finalizers:[]string(nil), ClusterName:"", ManagedFields:[]v1.ManagedFieldsEntry(nil)}, InvolvedObject:v1.ObjectReference{Kind:"Node", Namespace:"", Name:"config-control-plane", UID:"config-control-plane", APIVersion:"", ResourceVersion:"", FieldPath:""}, Reason:"Starting", Message:"Starting kubelet.", Source:v1.EventSource{Component:"kubelet", Host:"config-control-plane"}, FirstTimestamp:v1.Time{Time:time.Time{wall:0xc03f1b998dede9d4, ext:8748535501, loc:(*time.Location)(0x74aba00)}}, LastTimestamp:v1.Time{Time:time.Time{wall:0xc03f1b998dede9d4, ext:8748535501, loc:(*time.Location)(0x74aba00)}}, Count:1, Type:"Normal", EventTime:v1.MicroTime{Time:time.Time{wall:0x0, ext:0, loc:(*time.Location)(nil)}}, Series:(*v1.EventSeries)(nil), Action:"", Related:(*v1.ObjectReference)(nil), ReportingController:"", ReportingInstance:""}': 'namespaces "default" not found' (will not retry!)
Aug 17 17:50:40 config-control-plane kubelet[286]: I0817 17:50:40.933163     286 apiserver.go:52] "Watching apiserver"
Aug 17 17:50:40 config-control-plane kubelet[286]: E0817 17:50:40.955018     286 event.go:264] Server rejected event '&v1.Event{TypeMeta:v1.TypeMeta{Kind:"", APIVersion:""}, ObjectMeta:v1.ObjectMeta{Name:"config-control-plane.169c2959a97e3950", GenerateName:"", Namespace:"default", SelfLink:"", UID:"", ResourceVersion:"", Generation:0, CreationTimestamp:v1.Time{Time:time.Time{wall:0x0, ext:0, loc:(*time.Location)(nil)}}, DeletionTimestamp:(*v1.Time)(nil), DeletionGracePeriodSeconds:(*int64)(nil), Labels:map[string]string(nil), Annotations:map[string]string(nil), OwnerReferences:[]v1.OwnerReference(nil), Finalizers:[]string(nil), ClusterName:"", ManagedFields:[]v1.ManagedFieldsEntry(nil)}, InvolvedObject:v1.ObjectReference{Kind:"Node", Namespace:"", Name:"config-control-plane", UID:"config-control-plane", APIVersion:"", ResourceVersion:"", FieldPath:""}, Reason:"NodeHasSufficientMemory", Message:"Node config-control-plane status is now: NodeHasSufficientMemory", Source:v1.EventSource{Component:"kubelet", Host:"config-control-plane"}, FirstTimestamp:v1.Time{Time:time.Time{wall:0xc03f1b999050bd50, ext:8788565001, loc:(*time.Location)(0x74aba00)}}, LastTimestamp:v1.Time{Time:time.Time{wall:0xc03f1b999050bd50, ext:8788565001, loc:(*time.Location)(0x74aba00)}}, Count:1, Type:"Normal", EventTime:v1.MicroTime{Time:time.Time{wall:0x0, ext:0, loc:(*time.Location)(nil)}}, Series:(*v1.EventSeries)(nil), Action:"", Related:(*v1.ObjectReference)(nil), ReportingController:"", ReportingInstance:""}': 'namespaces "default" not found' (will not retry!)
Aug 17 17:50:41 config-control-plane kubelet[286]: E0817 17:50:41.009854     286 event.go:264] Server rejected event '&v1.Event{TypeMeta:v1.TypeMeta{Kind:"", APIVersion:""}, ObjectMeta:v1.ObjectMeta{Name:"config-control-plane.169c2959a97e98a0", GenerateName:"", Namespace:"default", SelfLink:"", UID:"", ResourceVersion:"", Generation:0, CreationTimestamp:v1.Time{Time:time.Time{wall:0x0, ext:0, loc:(*time.Location)(nil)}}, DeletionTimestamp:(*v1.Time)(nil), DeletionGracePeriodSeconds:(*int64)(nil), Labels:map[string]string(nil), Annotations:map[string]string(nil), OwnerReferences:[]v1.OwnerReference(nil), Finalizers:[]string(nil), ClusterName:"", ManagedFields:[]v1.ManagedFieldsEntry(nil)}, InvolvedObject:v1.ObjectReference{Kind:"Node", Namespace:"", Name:"config-control-plane", UID:"config-control-plane", APIVersion:"", ResourceVersion:"", FieldPath:""}, Reason:"NodeHasNoDiskPressure", Message:"Node config-control-plane status is now: NodeHasNoDiskPressure", Source:v1.EventSource{Component:"kubelet", Host:"config-control-plane"}, FirstTimestamp:v1.Time{Time:time.Time{wall:0xc03f1b9990511ca0, ext:8788588901, loc:(*time.Location)(0x74aba00)}}, LastTimestamp:v1.Time{Time:time.Time{wall:0xc03f1b9990511ca0, ext:8788588901, loc:(*time.Location)(0x74aba00)}}, Count:1, Type:"Normal", EventTime:v1.MicroTime{Time:time.Time{wall:0x0, ext:0, loc:(*time.Location)(nil)}}, Series:(*v1.EventSeries)(nil), Action:"", Related:(*v1.ObjectReference)(nil), ReportingController:"", ReportingInstance:""}': 'namespaces "default" not found' (will not retry!)
Aug 17 17:50:41 config-control-plane kubelet[286]: I0817 17:50:41.024635     286 reconciler.go:157] "Reconciler: start to sync state"
Aug 17 17:50:41 config-control-plane kubelet[286]: E0817 17:50:41.064948     286 event.go:264] Server rejected event '&v1.Event{TypeMeta:v1.TypeMeta{Kind:"", APIVersion:""}, ObjectMeta:v1.ObjectMeta{Name:"config-control-plane.169c2959a97ed7e8", GenerateName:"", Namespace:"default", SelfLink:"", UID:"", ResourceVersion:"", Generation:0, CreationTimestamp:v1.Time{Time:time.Time{wall:0x0, ext:0, loc:(*time.Location)(nil)}}, DeletionTimestamp:(*v1.Time)(nil), DeletionGracePeriodSeconds:(*int64)(nil), Labels:map[string]string(nil), Annotations:map[string]string(nil), OwnerReferences:[]v1.OwnerReference(nil), Finalizers:[]string(nil), ClusterName:"", ManagedFields:[]v1.ManagedFieldsEntry(nil)}, InvolvedObject:v1.ObjectReference{Kind:"Node", Namespace:"", Name:"config-control-plane", UID:"config-control-plane", APIVersion:"", ResourceVersion:"", FieldPath:""}, Reason:"NodeHasSufficientPID", Message:"Node config-control-plane status is now: NodeHasSufficientPID", Source:v1.EventSource{Component:"kubelet", Host:"config-control-plane"}, FirstTimestamp:v1.Time{Time:time.Time{wall:0xc03f1b9990515be8, ext:8788605001, loc:(*time.Location)(0x74aba00)}}, LastTimestamp:v1.Time{Time:time.Time{wall:0xc03f1b9990515be8, ext:8788605001, loc:(*time.Location)(0x74aba00)}}, Count:1, Type:"Normal", EventTime:v1.MicroTime{Time:time.Time{wall:0x0, ext:0, loc:(*time.Location)(nil)}}, Series:(*v1.EventSeries)(nil), Action:"", Related:(*v1.ObjectReference)(nil), ReportingController:"", ReportingInstance:""}': 'namespaces "default" not found' (will not retry!)
Aug 17 17:50:41 config-control-plane kubelet[286]: E0817 17:50:41.120530     286 event.go:264] Server rejected event '&v1.Event{TypeMeta:v1.TypeMeta{Kind:"", APIVersion:""}, ObjectMeta:v1.ObjectMeta{Name:"config-control-plane.169c2959aa711f58", GenerateName:"", Namespace:"default", SelfLink:"", UID:"", ResourceVersion:"", Generation:0, CreationTimestamp:v1.Time{Time:time.Time{wall:0x0, ext:0, loc:(*time.Location)(nil)}}, DeletionTimestamp:(*v1.Time)(nil), DeletionGracePeriodSeconds:(*int64)(nil), Labels:map[string]string(nil), Annotations:map[string]string(nil), OwnerReferences:[]v1.OwnerReference(nil), Finalizers:[]string(nil), ClusterName:"", ManagedFields:[]v1.ManagedFieldsEntry(nil)}, InvolvedObject:v1.ObjectReference{Kind:"Node", Namespace:"", Name:"config-control-plane", UID:"config-control-plane", APIVersion:"", ResourceVersion:"", FieldPath:""}, Reason:"NodeAllocatableEnforced", Message:"Updated Node Allocatable limit across pods", Source:v1.EventSource{Component:"kubelet", Host:"config-control-plane"}, FirstTimestamp:v1.Time{Time:time.Time{wall:0xc03f1b999143a358, ext:8804484201, loc:(*time.Location)(0x74aba00)}}, LastTimestamp:v1.Time{Time:time.Time{wall:0xc03f1b999143a358, ext:8804484201, loc:(*time.Location)(0x74aba00)}}, Count:1, Type:"Normal", EventTime:v1.MicroTime{Time:time.Time{wall:0x0, ext:0, loc:(*time.Location)(nil)}}, Series:(*v1.EventSeries)(nil), Action:"", Related:(*v1.ObjectReference)(nil), ReportingController:"", ReportingInstance:""}': 'namespaces "default" not found' (will not retry!)
Aug 17 17:50:41 config-control-plane kubelet[286]: E0817 17:50:41.178213     286 event.go:264] Server rejected event '&v1.Event{TypeMeta:v1.TypeMeta{Kind:"", APIVersion:""}, ObjectMeta:v1.ObjectMeta{Name:"config-control-plane.169c2959a97e3950", GenerateName:"", Namespace:"default", SelfLink:"", UID:"", ResourceVersion:"", Generation:0, CreationTimestamp:v1.Time{Time:time.Time{wall:0x0, ext:0, loc:(*time.Location)(nil)}}, DeletionTimestamp:(*v1.Time)(nil), DeletionGracePeriodSeconds:(*int64)(nil), Labels:map[string]string(nil), Annotations:map[string]string(nil), OwnerReferences:[]v1.OwnerReference(nil), Finalizers:[]string(nil), ClusterName:"", ManagedFields:[]v1.ManagedFieldsEntry(nil)}, InvolvedObject:v1.ObjectReference{Kind:"Node", Namespace:"", Name:"config-control-plane", UID:"config-control-plane", APIVersion:"", ResourceVersion:"", FieldPath:""}, Reason:"NodeHasSufficientMemory", Message:"Node config-control-plane status is now: NodeHasSufficientMemory", Source:v1.EventSource{Component:"kubelet", Host:"config-control-plane"}, FirstTimestamp:v1.Time{Time:time.Time{wall:0xc03f1b999050bd50, ext:8788565001, loc:(*time.Location)(0x74aba00)}}, LastTimestamp:v1.Time{Time:time.Time{wall:0xc03f1b99943d9780, ext:8854419001, loc:(*time.Location)(0x74aba00)}}, Count:2, Type:"Normal", EventTime:v1.MicroTime{Time:time.Time{wall:0x0, ext:0, loc:(*time.Location)(nil)}}, Series:(*v1.EventSeries)(nil), Action:"", Related:(*v1.ObjectReference)(nil), ReportingController:"", ReportingInstance:""}': 'namespaces "default" not found' (will not retry!)
Aug 17 17:50:41 config-control-plane kubelet[286]: E0817 17:50:41.236728     286 event.go:264] Server rejected event '&v1.Event{TypeMeta:v1.TypeMeta{Kind:"", APIVersion:""}, ObjectMeta:v1.ObjectMeta{Name:"config-control-plane.169c2959a97e98a0", GenerateName:"", Namespace:"default", SelfLink:"", UID:"", ResourceVersion:"", Generation:0, CreationTimestamp:v1.Time{Time:time.Time{wall:0x0, ext:0, loc:(*time.Location)(nil)}}, DeletionTimestamp:(*v1.Time)(nil), DeletionGracePeriodSeconds:(*int64)(nil), Labels:map[string]string(nil), Annotations:map[string]string(nil), OwnerReferences:[]v1.OwnerReference(nil), Finalizers:[]string(nil), ClusterName:"", ManagedFields:[]v1.ManagedFieldsEntry(nil)}, InvolvedObject:v1.ObjectReference{Kind:"Node", Namespace:"", Name:"config-control-plane", UID:"config-control-plane", APIVersion:"", ResourceVersion:"", FieldPath:""}, Reason:"NodeHasNoDiskPressure", Message:"Node config-control-plane status is now: NodeHasNoDiskPressure", Source:v1.EventSource{Component:"kubelet", Host:"config-control-plane"}, FirstTimestamp:v1.Time{Time:time.Time{wall:0xc03f1b9990511ca0, ext:8788588901, loc:(*time.Location)(0x74aba00)}}, LastTimestamp:v1.Time{Time:time.Time{wall:0xc03f1b99943de474, ext:8854438101, loc:(*time.Location)(0x74aba00)}}, Count:2, Type:"Normal", EventTime:v1.MicroTime{Time:time.Time{wall:0x0, ext:0, loc:(*time.Location)(nil)}}, Series:(*v1.EventSeries)(nil), Action:"", Related:(*v1.ObjectReference)(nil), ReportingController:"", ReportingInstance:""}': 'namespaces "default" not found' (will not retry!)
Aug 17 17:50:41 config-control-plane kubelet[286]: E0817 17:50:41.294538     286 event.go:264] Server rejected event '&v1.Event{TypeMeta:v1.TypeMeta{Kind:"", APIVersion:""}, ObjectMeta:v1.ObjectMeta{Name:"config-control-plane.169c2959a97ed7e8", GenerateName:"", Namespace:"default", SelfLink:"", UID:"", ResourceVersion:"", Generation:0, CreationTimestamp:v1.Time{Time:time.Time{wall:0x0, ext:0, loc:(*time.Location)(nil)}}, DeletionTimestamp:(*v1.Time)(nil), DeletionGracePeriodSeconds:(*int64)(nil), Labels:map[string]string(nil), Annotations:map[string]string(nil), OwnerReferences:[]v1.OwnerReference(nil), Finalizers:[]string(nil), ClusterName:"", ManagedFields:[]v1.ManagedFieldsEntry(nil)}, InvolvedObject:v1.ObjectReference{Kind:"Node", Namespace:"", Name:"config-control-plane", UID:"config-control-plane", APIVersion:"", ResourceVersion:"", FieldPath:""}, Reason:"NodeHasSufficientPID", Message:"Node config-control-plane status is now: NodeHasSufficientPID", Source:v1.EventSource{Component:"kubelet", Host:"config-control-plane"}, FirstTimestamp:v1.Time{Time:time.Time{wall:0xc03f1b9990515be8, ext:8788605001, loc:(*time.Location)(0x74aba00)}}, LastTimestamp:v1.Time{Time:time.Time{wall:0xc03f1b99943e2740, ext:8854455401, loc:(*time.Location)(0x74aba00)}}, Count:2, Type:"Normal", EventTime:v1.MicroTime{Time:time.Time{wall:0x0, ext:0, loc:(*time.Location)(nil)}}, Series:(*v1.EventSeries)(nil), Action:"", Related:(*v1.ObjectReference)(nil), ReportingController:"", ReportingInstance:""}': 'namespaces "default" not found' (will not retry!)
Aug 17 17:50:41 config-control-plane kubelet[286]: E0817 17:50:41.357203     286 event.go:264] Server rejected event '&v1.Event{TypeMeta:v1.TypeMeta{Kind:"", APIVersion:""}, ObjectMeta:v1.ObjectMeta{Name:"config-control-plane.169c2959a97e3950", GenerateName:"", Namespace:"default", SelfLink:"", UID:"", ResourceVersion:"", Generation:0, CreationTimestamp:v1.Time{Time:time.Time{wall:0x0, ext:0, loc:(*time.Location)(nil)}}, DeletionTimestamp:(*v1.Time)(nil), DeletionGracePeriodSeconds:(*int64)(nil), Labels:map[string]string(nil), Annotations:map[string]string(nil), OwnerReferences:[]v1.OwnerReference(nil), Finalizers:[]string(nil), ClusterName:"", ManagedFields:[]v1.ManagedFieldsEntry(nil)}, InvolvedObject:v1.ObjectReference{Kind:"Node", Namespace:"", Name:"config-control-plane", UID:"config-control-plane", APIVersion:"", ResourceVersion:"", FieldPath:""}, Reason:"NodeHasSufficientMemory", Message:"Node config-control-plane status is now: NodeHasSufficientMemory", Source:v1.EventSource{Component:"kubelet", Host:"config-control-plane"}, FirstTimestamp:v1.Time{Time:time.Time{wall:0xc03f1b999050bd50, ext:8788565001, loc:(*time.Location)(0x74aba00)}}, LastTimestamp:v1.Time{Time:time.Time{wall:0xc03f1b99a04cd964, ext:9056745301, loc:(*time.Location)(0x74aba00)}}, Count:3, Type:"Normal", EventTime:v1.MicroTime{Time:time.Time{wall:0x0, ext:0, loc:(*time.Location)(nil)}}, Series:(*v1.EventSeries)(nil), Action:"", Related:(*v1.ObjectReference)(nil), ReportingController:"", ReportingInstance:""}': 'namespaces "default" not found' (will not retry!)
Aug 17 17:50:41 config-control-plane kubelet[286]: E0817 17:50:41.758240     286 event.go:264] Server rejected event '&v1.Event{TypeMeta:v1.TypeMeta{Kind:"", APIVersion:""}, ObjectMeta:v1.ObjectMeta{Name:"config-control-plane.169c2959a97e98a0", GenerateName:"", Namespace:"default", SelfLink:"", UID:"", ResourceVersion:"", Generation:0, CreationTimestamp:v1.Time{Time:time.Time{wall:0x0, ext:0, loc:(*time.Location)(nil)}}, DeletionTimestamp:(*v1.Time)(nil), DeletionGracePeriodSeconds:(*int64)(nil), Labels:map[string]string(nil), Annotations:map[string]string(nil), OwnerReferences:[]v1.OwnerReference(nil), Finalizers:[]string(nil), ClusterName:"", ManagedFields:[]v1.ManagedFieldsEntry(nil)}, InvolvedObject:v1.ObjectReference{Kind:"Node", Namespace:"", Name:"config-control-plane", UID:"config-control-plane", APIVersion:"", ResourceVersion:"", FieldPath:""}, Reason:"NodeHasNoDiskPressure", Message:"Node config-control-plane status is now: NodeHasNoDiskPressure", Source:v1.EventSource{Component:"kubelet", Host:"config-control-plane"}, FirstTimestamp:v1.Time{Time:time.Time{wall:0xc03f1b9990511ca0, ext:8788588901, loc:(*time.Location)(0x74aba00)}}, LastTimestamp:v1.Time{Time:time.Time{wall:0xc03f1b99a04d2e28, ext:9056766401, loc:(*time.Location)(0x74aba00)}}, Count:3, Type:"Normal", EventTime:v1.MicroTime{Time:time.Time{wall:0x0, ext:0, loc:(*time.Location)(nil)}}, Series:(*v1.EventSeries)(nil), Action:"", Related:(*v1.ObjectReference)(nil), ReportingController:"", ReportingInstance:""}': 'namespaces "default" not found' (will not retry!)
Aug 17 17:50:42 config-control-plane kubelet[286]: E0817 17:50:42.156853     286 event.go:264] Server rejected event '&v1.Event{TypeMeta:v1.TypeMeta{Kind:"", APIVersion:""}, ObjectMeta:v1.ObjectMeta{Name:"config-control-plane.169c2959a97ed7e8", GenerateName:"", Namespace:"default", SelfLink:"", UID:"", ResourceVersion:"", Generation:0, CreationTimestamp:v1.Time{Time:time.Time{wall:0x0, ext:0, loc:(*time.Location)(nil)}}, DeletionTimestamp:(*v1.Time)(nil), DeletionGracePeriodSeconds:(*int64)(nil), Labels:map[string]string(nil), Annotations:map[string]string(nil), OwnerReferences:[]v1.OwnerReference(nil), Finalizers:[]string(nil), ClusterName:"", ManagedFields:[]v1.ManagedFieldsEntry(nil)}, InvolvedObject:v1.ObjectReference{Kind:"Node", Namespace:"", Name:"config-control-plane", UID:"config-control-plane", APIVersion:"", ResourceVersion:"", FieldPath:""}, Reason:"NodeHasSufficientPID", Message:"Node config-control-plane status is now: NodeHasSufficientPID", Source:v1.EventSource{Component:"kubelet", Host:"config-control-plane"}, FirstTimestamp:v1.Time{Time:time.Time{wall:0xc03f1b9990515be8, ext:8788605001, loc:(*time.Location)(0x74aba00)}}, LastTimestamp:v1.Time{Time:time.Time{wall:0xc03f1b99a04d7d74, ext:9056786701, loc:(*time.Location)(0x74aba00)}}, Count:3, Type:"Normal", EventTime:v1.MicroTime{Time:time.Time{wall:0x0, ext:0, loc:(*time.Location)(nil)}}, Series:(*v1.EventSeries)(nil), Action:"", Related:(*v1.ObjectReference)(nil), ReportingController:"", ReportingInstance:""}': 'namespaces "default" not found' (will not retry!)
Aug 17 17:50:42 config-control-plane kubelet[286]: E0817 17:50:42.556180     286 event.go:264] Server rejected event '&v1.Event{TypeMeta:v1.TypeMeta{Kind:"", APIVersion:""}, ObjectMeta:v1.ObjectMeta{Name:"config-control-plane.169c2959a97e3950", GenerateName:"", Namespace:"default", SelfLink:"", UID:"", ResourceVersion:"", Generation:0, CreationTimestamp:v1.Time{Time:time.Time{wall:0x0, ext:0, loc:(*time.Location)(nil)}}, DeletionTimestamp:(*v1.Time)(nil), DeletionGracePeriodSeconds:(*int64)(nil), Labels:map[string]string(nil), Annotations:map[string]string(nil), OwnerReferences:[]v1.OwnerReference(nil), Finalizers:[]string(nil), ClusterName:"", ManagedFields:[]v1.ManagedFieldsEntry(nil)}, InvolvedObject:v1.ObjectReference{Kind:"Node", Namespace:"", Name:"config-control-plane", UID:"config-control-plane", APIVersion:"", ResourceVersion:"", FieldPath:""}, Reason:"NodeHasSufficientMemory", Message:"Node config-control-plane status is now: NodeHasSufficientMemory", Source:v1.EventSource{Component:"kubelet", Host:"config-control-plane"}, FirstTimestamp:v1.Time{Time:time.Time{wall:0xc03f1b999050bd50, ext:8788565001, loc:(*time.Location)(0x74aba00)}}, LastTimestamp:v1.Time{Time:time.Time{wall:0xc03f1b99b850a810, ext:9459649701, loc:(*time.Location)(0x74aba00)}}, Count:4, Type:"Normal", EventTime:v1.MicroTime{Time:time.Time{wall:0x0, ext:0, loc:(*time.Location)(nil)}}, Series:(*v1.EventSeries)(nil), Action:"", Related:(*v1.ObjectReference)(nil), ReportingController:"", ReportingInstance:""}': 'namespaces "default" not found' (will not retry!)
Aug 17 17:50:44 config-control-plane systemd[1]: Stopping kubelet: The Kubernetes Node Agent...
Aug 17 17:50:44 config-control-plane kubelet[286]: I0817 17:50:44.179511     286 dynamic_cafile_content.go:182] Shutting down client-ca-bundle::/etc/kubernetes/pki/ca.crt
Aug 17 17:50:44 config-control-plane systemd[1]: kubelet.service: Succeeded.
Aug 17 17:50:44 config-control-plane systemd[1]: Stopped kubelet: The Kubernetes Node Agent.
Aug 17 17:50:44 config-control-plane systemd[1]: Starting kubelet: The Kubernetes Node Agent...
Aug 17 17:50:44 config-control-plane systemd[1]: Started kubelet: The Kubernetes Node Agent.
Aug 17 17:50:44 config-control-plane kubelet[773]: Flag --fail-swap-on has been deprecated, This parameter should be set via the config file specified by the Kubelet's --config flag. See https://kubernetes.io/docs/tasks/administer-cluster/kubelet-config-file/ for more information.
Aug 17 17:50:44 config-control-plane kubelet[773]: Flag --provider-id has been deprecated, This parameter should be set via the config file specified by the Kubelet's --config flag. See https://kubernetes.io/docs/tasks/administer-cluster/kubelet-config-file/ for more information.
Aug 17 17:50:44 config-control-plane kubelet[773]: Flag --fail-swap-on has been deprecated, This parameter should be set via the config file specified by the Kubelet's --config flag. See https://kubernetes.io/docs/tasks/administer-cluster/kubelet-config-file/ for more information.
Aug 17 17:50:44 config-control-plane kubelet[773]: Flag --cgroup-root has been deprecated, This parameter should be set via the config file specified by the Kubelet's --config flag. See https://kubernetes.io/docs/tasks/administer-cluster/kubelet-config-file/ for more information.
Aug 17 17:50:44 config-control-plane kubelet[773]: I0817 17:50:44.324615     773 server.go:197] "Warning: For remote container runtime, --pod-infra-container-image is ignored in kubelet, which should be set in that remote runtime instead"
Aug 17 17:50:44 config-control-plane kubelet[773]: Flag --fail-swap-on has been deprecated, This parameter should be set via the config file specified by the Kubelet's --config flag. See https://kubernetes.io/docs/tasks/administer-cluster/kubelet-config-file/ for more information.
Aug 17 17:50:44 config-control-plane kubelet[773]: Flag --provider-id has been deprecated, This parameter should be set via the config file specified by the Kubelet's --config flag. See https://kubernetes.io/docs/tasks/administer-cluster/kubelet-config-file/ for more information.
Aug 17 17:50:44 config-control-plane kubelet[773]: Flag --fail-swap-on has been deprecated, This parameter should be set via the config file specified by the Kubelet's --config flag. See https://kubernetes.io/docs/tasks/administer-cluster/kubelet-config-file/ for more information.
Aug 17 17:50:44 config-control-plane kubelet[773]: Flag --cgroup-root has been deprecated, This parameter should be set via the config file specified by the Kubelet's --config flag. See https://kubernetes.io/docs/tasks/administer-cluster/kubelet-config-file/ for more information.
Aug 17 17:50:44 config-control-plane kubelet[773]: I0817 17:50:44.337616     773 server.go:440] "Kubelet version" kubeletVersion="v1.21.1"
Aug 17 17:50:44 config-control-plane kubelet[773]: I0817 17:50:44.337927     773 server.go:851] "Client rotation is on, will bootstrap in background"
Aug 17 17:50:44 config-control-plane kubelet[773]: I0817 17:50:44.339795     773 certificate_store.go:130] Loading cert/key pair from "/var/lib/kubelet/pki/kubelet-client-current.pem".
Aug 17 17:50:44 config-control-plane kubelet[773]: I0817 17:50:44.340868     773 dynamic_cafile_content.go:167] Starting client-ca-bundle::/etc/kubernetes/pki/ca.crt
Aug 17 17:50:49 config-control-plane kubelet[773]: W0817 17:50:49.351172     773 sysinfo.go:203] Nodes topology is not available, providing CPU topology
Aug 17 17:50:49 config-control-plane kubelet[773]: I0817 17:50:49.356386     773 container_manager_linux.go:278] "Container manager verified user specified cgroup-root exists" cgroupRoot=[kubelet]
Aug 17 17:50:49 config-control-plane kubelet[773]: I0817 17:50:49.356462     773 container_manager_linux.go:283] "Creating Container Manager object based on Node Config" nodeConfig={RuntimeCgroupsName: SystemCgroupsName: KubeletCgroupsName: ContainerRuntime:remote CgroupsPerQOS:true CgroupRoot:/kubelet CgroupDriver:cgroupfs KubeletRootDir:/var/lib/kubelet ProtectKernelDefaults:false NodeAllocatableConfig:{KubeReservedCgroupName: SystemReservedCgroupName: ReservedSystemCPUs: EnforceNodeAllocatable:map[pods:{}] KubeReserved:map[] SystemReserved:map[] HardEvictionThresholds:[]} QOSReserved:map[] ExperimentalCPUManagerPolicy:none ExperimentalTopologyManagerScope:container ExperimentalCPUManagerReconcilePeriod:10s ExperimentalMemoryManagerPolicy:None ExperimentalMemoryManagerReservedMemory:[] ExperimentalPodPidsLimit:-1 EnforceCPULimits:true CPUCFSQuotaPeriod:100ms ExperimentalTopologyManagerPolicy:none}
Aug 17 17:50:49 config-control-plane kubelet[773]: I0817 17:50:49.356590     773 topology_manager.go:120] "Creating topology manager with policy per scope" topologyPolicyName="none" topologyScopeName="container"
Aug 17 17:50:49 config-control-plane kubelet[773]: I0817 17:50:49.356614     773 container_manager_linux.go:314] "Initializing Topology Manager" policy="none" scope="container"
Aug 17 17:50:49 config-control-plane kubelet[773]: I0817 17:50:49.356631     773 container_manager_linux.go:319] "Creating device plugin manager" devicePluginEnabled=true
Aug 17 17:50:49 config-control-plane kubelet[773]: I0817 17:50:49.356775     773 remote_runtime.go:62] parsed scheme: ""
Aug 17 17:50:49 config-control-plane kubelet[773]: I0817 17:50:49.356801     773 remote_runtime.go:62] scheme "" not registered, fallback to default scheme
Aug 17 17:50:49 config-control-plane kubelet[773]: I0817 17:50:49.356841     773 passthrough.go:48] ccResolverWrapper: sending update to cc: {[{/run/containerd/containerd.sock  <nil> 0 <nil>}] <nil> <nil>}
Aug 17 17:50:49 config-control-plane kubelet[773]: I0817 17:50:49.356862     773 clientconn.go:948] ClientConn switching balancer to "pick_first"
Aug 17 17:50:49 config-control-plane kubelet[773]: I0817 17:50:49.356926     773 remote_image.go:50] parsed scheme: ""
Aug 17 17:50:49 config-control-plane kubelet[773]: I0817 17:50:49.356940     773 remote_image.go:50] scheme "" not registered, fallback to default scheme
Aug 17 17:50:49 config-control-plane kubelet[773]: I0817 17:50:49.356961     773 passthrough.go:48] ccResolverWrapper: sending update to cc: {[{/run/containerd/containerd.sock  <nil> 0 <nil>}] <nil> <nil>}
Aug 17 17:50:49 config-control-plane kubelet[773]: I0817 17:50:49.356976     773 clientconn.go:948] ClientConn switching balancer to "pick_first"
Aug 17 17:50:49 config-control-plane kubelet[773]: I0817 17:50:49.357041     773 kubelet.go:404] "Attempting to sync node with API server"
Aug 17 17:50:49 config-control-plane kubelet[773]: I0817 17:50:49.357072     773 kubelet.go:272] "Adding static pod path" path="/etc/kubernetes/manifests"
Aug 17 17:50:49 config-control-plane kubelet[773]: I0817 17:50:49.357111     773 kubelet.go:283] "Adding apiserver pod source"
Aug 17 17:50:49 config-control-plane kubelet[773]: I0817 17:50:49.357136     773 apiserver.go:42] "Waiting for node sync before watching apiserver pods"
Aug 17 17:50:49 config-control-plane kubelet[773]: I0817 17:50:49.357636     773 clientconn.go:897] blockingPicker: the picked transport is not ready, loop back to repick
Aug 17 17:50:49 config-control-plane kubelet[773]: I0817 17:50:49.359956     773 kuberuntime_manager.go:222] "Container runtime initialized" containerRuntime="containerd" version="v1.5.1" apiVersion="v1alpha2"
Aug 17 17:50:50 config-control-plane kubelet[773]: I0817 17:50:50.358992     773 apiserver.go:52] "Watching apiserver"
Aug 17 17:50:54 config-control-plane kubelet[773]: E0817 17:50:54.625568     773 aws_credentials.go:77] while getting AWS credentials NoCredentialProviders: no valid providers in chain. Deprecated.
Aug 17 17:50:54 config-control-plane kubelet[773]:         For verbose messaging see aws.Config.CredentialsChainVerboseErrors
Aug 17 17:50:54 config-control-plane kubelet[773]: I0817 17:50:54.627071     773 server.go:1190] "Started kubelet"
Aug 17 17:50:54 config-control-plane kubelet[773]: I0817 17:50:54.628115     773 server.go:149] "Starting to listen" address="0.0.0.0" port=10250
Aug 17 17:50:54 config-control-plane kubelet[773]: I0817 17:50:54.629060     773 server.go:405] "Adding debug handlers to kubelet server"
Aug 17 17:50:54 config-control-plane kubelet[773]: I0817 17:50:54.631596     773 fs_resource_analyzer.go:67] "Starting FS ResourceAnalyzer"
Aug 17 17:50:54 config-control-plane kubelet[773]: I0817 17:50:54.632606     773 volume_manager.go:271] "Starting Kubelet Volume Manager"
Aug 17 17:50:54 config-control-plane kubelet[773]: I0817 17:50:54.632744     773 desired_state_of_world_populator.go:141] "Desired state populator starts to run"
Aug 17 17:50:54 config-control-plane kubelet[773]: E0817 17:50:54.643040     773 kubelet.go:2211] "Container runtime network not ready" networkReady="NetworkReady=false reason:NetworkPluginNotReady message:Network plugin returns error: cni plugin not initialized"
Aug 17 17:50:54 config-control-plane kubelet[773]: I0817 17:50:54.643314     773 client.go:86] parsed scheme: "unix"
Aug 17 17:50:54 config-control-plane kubelet[773]: I0817 17:50:54.643336     773 client.go:86] scheme "unix" not registered, fallback to default scheme
Aug 17 17:50:54 config-control-plane kubelet[773]: I0817 17:50:54.643394     773 passthrough.go:48] ccResolverWrapper: sending update to cc: {[{unix:///run/containerd/containerd.sock  <nil> 0 <nil>}] <nil> <nil>}
Aug 17 17:50:54 config-control-plane kubelet[773]: I0817 17:50:54.643417     773 clientconn.go:948] ClientConn switching balancer to "pick_first"
Aug 17 17:50:54 config-control-plane kubelet[773]: I0817 17:50:54.737363     773 kubelet_node_status.go:71] "Attempting to register node" node="config-control-plane"
Aug 17 17:50:54 config-control-plane kubelet[773]: I0817 17:50:54.754297     773 kubelet_node_status.go:109] "Node was previously registered" node="config-control-plane"
Aug 17 17:50:54 config-control-plane kubelet[773]: I0817 17:50:54.754585     773 kubelet_node_status.go:74] "Successfully registered node" node="config-control-plane"
Aug 17 17:50:54 config-control-plane kubelet[773]: I0817 17:50:54.867633     773 kubelet_network_linux.go:56] "Initialized protocol iptables rules." protocol=IPv4
Aug 17 17:50:54 config-control-plane kubelet[773]: I0817 17:50:54.953032     773 cpu_manager.go:199] "Starting CPU manager" policy="none"
Aug 17 17:50:54 config-control-plane kubelet[773]: I0817 17:50:54.953349     773 cpu_manager.go:200] "Reconciling" reconcilePeriod="10s"
Aug 17 17:50:54 config-control-plane kubelet[773]: I0817 17:50:54.953528     773 state_mem.go:36] "Initialized new in-memory state store"
Aug 17 17:50:54 config-control-plane kubelet[773]: I0817 17:50:54.953770     773 state_mem.go:88] "Updated default CPUSet" cpuSet=""
Aug 17 17:50:54 config-control-plane kubelet[773]: I0817 17:50:54.953902     773 state_mem.go:96] "Updated CPUSet assignments" assignments=map[]
Aug 17 17:50:54 config-control-plane kubelet[773]: I0817 17:50:54.954031     773 policy_none.go:44] "None policy: Start"
Aug 17 17:50:54 config-control-plane kubelet[773]: I0817 17:50:54.955358     773 manager.go:600] "Failed to retrieve checkpoint" checkpoint="kubelet_internal_checkpoint" err="checkpoint is not found"
Aug 17 17:50:54 config-control-plane kubelet[773]: I0817 17:50:54.955775     773 plugin_manager.go:114] "Starting Kubelet Plugin Manager"
Aug 17 17:50:55 config-control-plane kubelet[773]: I0817 17:50:55.029466     773 kubelet_network_linux.go:56] "Initialized protocol iptables rules." protocol=IPv6
Aug 17 17:50:55 config-control-plane kubelet[773]: I0817 17:50:55.029525     773 status_manager.go:157] "Starting to sync pod status with apiserver"
Aug 17 17:50:55 config-control-plane kubelet[773]: I0817 17:50:55.029574     773 kubelet.go:1846] "Starting kubelet main sync loop"
Aug 17 17:50:55 config-control-plane kubelet[773]: E0817 17:50:55.029657     773 kubelet.go:1870] "Skipping pod synchronization" err="PLEG is not healthy: pleg has yet to be successful"
Aug 17 17:50:55 config-control-plane kubelet[773]: I0817 17:50:55.129965     773 topology_manager.go:187] "Topology Admit Handler"
Aug 17 17:50:55 config-control-plane kubelet[773]: I0817 17:50:55.130146     773 topology_manager.go:187] "Topology Admit Handler"
Aug 17 17:50:55 config-control-plane kubelet[773]: I0817 17:50:55.130297     773 topology_manager.go:187] "Topology Admit Handler"
Aug 17 17:50:55 config-control-plane kubelet[773]: I0817 17:50:55.130441     773 topology_manager.go:187] "Topology Admit Handler"
Aug 17 17:50:55 config-control-plane kubelet[773]: I0817 17:50:55.139669     773 reconciler.go:224] "operationExecutor.VerifyControllerAttachedVolume started for volume \"usr-share-ca-certificates\" (UniqueName: \"kubernetes.io/host-path/bd0bc82861de35a27312a75b3201e56c-usr-share-ca-certificates\") pod \"kube-controller-manager-config-control-plane\" (UID: \"bd0bc82861de35a27312a75b3201e56c\") "
Aug 17 17:50:55 config-control-plane kubelet[773]: I0817 17:50:55.139765     773 reconciler.go:224] "operationExecutor.VerifyControllerAttachedVolume started for volume \"kubeconfig\" (UniqueName: \"kubernetes.io/host-path/18ab771e4cf3f5b60acc165ef57a754d-kubeconfig\") pod \"kube-scheduler-config-control-plane\" (UID: \"18ab771e4cf3f5b60acc165ef57a754d\") "
Aug 17 17:50:55 config-control-plane kubelet[773]: I0817 17:50:55.139822     773 reconciler.go:224] "operationExecutor.VerifyControllerAttachedVolume started for volume \"etcd-certs\" (UniqueName: \"kubernetes.io/host-path/72c47c236abf4280d9f8b6af5ec1850d-etcd-certs\") pod \"etcd-config-control-plane\" (UID: \"72c47c236abf4280d9f8b6af5ec1850d\") "
Aug 17 17:50:55 config-control-plane kubelet[773]: I0817 17:50:55.139871     773 reconciler.go:224] "operationExecutor.VerifyControllerAttachedVolume started for volume \"etc-ca-certificates\" (UniqueName: \"kubernetes.io/host-path/6f9f8d44cbdd820f10746e0421ee018f-etc-ca-certificates\") pod \"kube-apiserver-config-control-plane\" (UID: \"6f9f8d44cbdd820f10746e0421ee018f\") "
Aug 17 17:50:55 config-control-plane kubelet[773]: I0817 17:50:55.139927     773 reconciler.go:224] "operationExecutor.VerifyControllerAttachedVolume started for volume \"usr-local-share-ca-certificates\" (UniqueName: \"kubernetes.io/host-path/6f9f8d44cbdd820f10746e0421ee018f-usr-local-share-ca-certificates\") pod \"kube-apiserver-config-control-plane\" (UID: \"6f9f8d44cbdd820f10746e0421ee018f\") "
Aug 17 17:50:55 config-control-plane kubelet[773]: I0817 17:50:55.140002     773 reconciler.go:224] "operationExecutor.VerifyControllerAttachedVolume started for volume \"ca-certs\" (UniqueName: \"kubernetes.io/host-path/bd0bc82861de35a27312a75b3201e56c-ca-certs\") pod \"kube-controller-manager-config-control-plane\" (UID: \"bd0bc82861de35a27312a75b3201e56c\") "
Aug 17 17:50:55 config-control-plane kubelet[773]: I0817 17:50:55.140049     773 reconciler.go:224] "operationExecutor.VerifyControllerAttachedVolume started for volume \"k8s-certs\" (UniqueName: \"kubernetes.io/host-path/bd0bc82861de35a27312a75b3201e56c-k8s-certs\") pod \"kube-controller-manager-config-control-plane\" (UID: \"bd0bc82861de35a27312a75b3201e56c\") "
Aug 17 17:50:55 config-control-plane kubelet[773]: I0817 17:50:55.140094     773 reconciler.go:224] "operationExecutor.VerifyControllerAttachedVolume started for volume \"kubeconfig\" (UniqueName: \"kubernetes.io/host-path/bd0bc82861de35a27312a75b3201e56c-kubeconfig\") pod \"kube-controller-manager-config-control-plane\" (UID: \"bd0bc82861de35a27312a75b3201e56c\") "
Aug 17 17:50:55 config-control-plane kubelet[773]: I0817 17:50:55.140153     773 reconciler.go:224] "operationExecutor.VerifyControllerAttachedVolume started for volume \"etcd-data\" (UniqueName: \"kubernetes.io/host-path/72c47c236abf4280d9f8b6af5ec1850d-etcd-data\") pod \"etcd-config-control-plane\" (UID: \"72c47c236abf4280d9f8b6af5ec1850d\") "
Aug 17 17:50:55 config-control-plane kubelet[773]: I0817 17:50:55.140202     773 reconciler.go:224] "operationExecutor.VerifyControllerAttachedVolume started for volume \"etc-ca-certificates\" (UniqueName: \"kubernetes.io/host-path/bd0bc82861de35a27312a75b3201e56c-etc-ca-certificates\") pod \"kube-controller-manager-config-control-plane\" (UID: \"bd0bc82861de35a27312a75b3201e56c\") "
Aug 17 17:50:55 config-control-plane kubelet[773]: I0817 17:50:55.140322     773 reconciler.go:224] "operationExecutor.VerifyControllerAttachedVolume started for volume \"ca-certs\" (UniqueName: \"kubernetes.io/host-path/6f9f8d44cbdd820f10746e0421ee018f-ca-certs\") pod \"kube-apiserver-config-control-plane\" (UID: \"6f9f8d44cbdd820f10746e0421ee018f\") "
Aug 17 17:50:55 config-control-plane kubelet[773]: I0817 17:50:55.140388     773 reconciler.go:224] "operationExecutor.VerifyControllerAttachedVolume started for volume \"usr-share-ca-certificates\" (UniqueName: \"kubernetes.io/host-path/6f9f8d44cbdd820f10746e0421ee018f-usr-share-ca-certificates\") pod \"kube-apiserver-config-control-plane\" (UID: \"6f9f8d44cbdd820f10746e0421ee018f\") "
Aug 17 17:50:55 config-control-plane kubelet[773]: I0817 17:50:55.140438     773 reconciler.go:224] "operationExecutor.VerifyControllerAttachedVolume started for volume \"flexvolume-dir\" (UniqueName: \"kubernetes.io/host-path/bd0bc82861de35a27312a75b3201e56c-flexvolume-dir\") pod \"kube-controller-manager-config-control-plane\" (UID: \"bd0bc82861de35a27312a75b3201e56c\") "
Aug 17 17:50:55 config-control-plane kubelet[773]: I0817 17:50:55.140493     773 reconciler.go:224] "operationExecutor.VerifyControllerAttachedVolume started for volume \"usr-local-share-ca-certificates\" (UniqueName: \"kubernetes.io/host-path/bd0bc82861de35a27312a75b3201e56c-usr-local-share-ca-certificates\") pod \"kube-controller-manager-config-control-plane\" (UID: \"bd0bc82861de35a27312a75b3201e56c\") "
Aug 17 17:50:55 config-control-plane kubelet[773]: I0817 17:50:55.140549     773 reconciler.go:224] "operationExecutor.VerifyControllerAttachedVolume started for volume \"k8s-certs\" (UniqueName: \"kubernetes.io/host-path/6f9f8d44cbdd820f10746e0421ee018f-k8s-certs\") pod \"kube-apiserver-config-control-plane\" (UID: \"6f9f8d44cbdd820f10746e0421ee018f\") "
Aug 17 17:50:55 config-control-plane kubelet[773]: I0817 17:50:55.140576     773 reconciler.go:157] "Reconciler: start to sync state"
Aug 17 17:50:55 config-control-plane kubelet[773]: E0817 17:50:55.144772     773 kubelet.go:1683] "Failed creating a mirror pod for" err="pods \"kube-apiserver-config-control-plane\" already exists" pod="kube-system/kube-apiserver-config-control-plane"
Aug 17 17:50:57 config-control-plane kubelet[773]: I0817 17:50:57.859560     773 topology_manager.go:187] "Topology Admit Handler"
Aug 17 17:50:57 config-control-plane kubelet[773]: I0817 17:50:57.862461     773 kuberuntime_manager.go:1044] "Updating runtime config through cri with podcidr" CIDR="10.20.0.0/24"
Aug 17 17:50:57 config-control-plane kubelet[773]: I0817 17:50:57.862732     773 reconciler.go:224] "operationExecutor.VerifyControllerAttachedVolume started for volume \"kube-proxy\" (UniqueName: \"kubernetes.io/configmap/4e49a535-a067-4256-8f9b-544d3bf3fff5-kube-proxy\") pod \"kube-proxy-qk46c\" (UID: \"4e49a535-a067-4256-8f9b-544d3bf3fff5\") "
Aug 17 17:50:57 config-control-plane kubelet[773]: I0817 17:50:57.862805     773 reconciler.go:224] "operationExecutor.VerifyControllerAttachedVolume started for volume \"kube-api-access-ctzsz\" (UniqueName: \"kubernetes.io/projected/4e49a535-a067-4256-8f9b-544d3bf3fff5-kube-api-access-ctzsz\") pod \"kube-proxy-qk46c\" (UID: \"4e49a535-a067-4256-8f9b-544d3bf3fff5\") "
Aug 17 17:50:57 config-control-plane kubelet[773]: I0817 17:50:57.862852     773 reconciler.go:224] "operationExecutor.VerifyControllerAttachedVolume started for volume \"xtables-lock\" (UniqueName: \"kubernetes.io/host-path/4e49a535-a067-4256-8f9b-544d3bf3fff5-xtables-lock\") pod \"kube-proxy-qk46c\" (UID: \"4e49a535-a067-4256-8f9b-544d3bf3fff5\") "
Aug 17 17:50:57 config-control-plane kubelet[773]: I0817 17:50:57.862892     773 reconciler.go:224] "operationExecutor.VerifyControllerAttachedVolume started for volume \"lib-modules\" (UniqueName: \"kubernetes.io/host-path/4e49a535-a067-4256-8f9b-544d3bf3fff5-lib-modules\") pod \"kube-proxy-qk46c\" (UID: \"4e49a535-a067-4256-8f9b-544d3bf3fff5\") "
Aug 17 17:50:57 config-control-plane kubelet[773]: I0817 17:50:57.864676     773 kubelet_network.go:76] "Updating Pod CIDR" originalPodCIDR="" newPodCIDR="10.20.0.0/24"
Aug 17 17:50:57 config-control-plane kubelet[773]: E0817 17:50:57.865263     773 kubelet.go:2211] "Container runtime network not ready" networkReady="NetworkReady=false reason:NetworkPluginNotReady message:Network plugin returns error: cni plugin not initialized"
Aug 17 17:50:57 config-control-plane kubelet[773]: I0817 17:50:57.878451     773 topology_manager.go:187] "Topology Admit Handler"
Aug 17 17:50:57 config-control-plane kubelet[773]: I0817 17:50:57.963285     773 reconciler.go:224] "operationExecutor.VerifyControllerAttachedVolume started for volume \"cni-cfg\" (UniqueName: \"kubernetes.io/host-path/9d57ef57-6889-486b-a24b-38c241f5ccec-cni-cfg\") pod \"kindnet-z8xnr\" (UID: \"9d57ef57-6889-486b-a24b-38c241f5ccec\") "
Aug 17 17:50:57 config-control-plane kubelet[773]: I0817 17:50:57.963546     773 reconciler.go:224] "operationExecutor.VerifyControllerAttachedVolume started for volume \"xtables-lock\" (UniqueName: \"kubernetes.io/host-path/9d57ef57-6889-486b-a24b-38c241f5ccec-xtables-lock\") pod \"kindnet-z8xnr\" (UID: \"9d57ef57-6889-486b-a24b-38c241f5ccec\") "
Aug 17 17:50:57 config-control-plane kubelet[773]: I0817 17:50:57.963649     773 reconciler.go:224] "operationExecutor.VerifyControllerAttachedVolume started for volume \"lib-modules\" (UniqueName: \"kubernetes.io/host-path/9d57ef57-6889-486b-a24b-38c241f5ccec-lib-modules\") pod \"kindnet-z8xnr\" (UID: \"9d57ef57-6889-486b-a24b-38c241f5ccec\") "
Aug 17 17:50:57 config-control-plane kubelet[773]: I0817 17:50:57.963698     773 reconciler.go:224] "operationExecutor.VerifyControllerAttachedVolume started for volume \"kube-api-access-5t886\" (UniqueName: \"kubernetes.io/projected/9d57ef57-6889-486b-a24b-38c241f5ccec-kube-api-access-5t886\") pod \"kindnet-z8xnr\" (UID: \"9d57ef57-6889-486b-a24b-38c241f5ccec\") "
Aug 17 17:50:59 config-control-plane kubelet[773]: E0817 17:50:59.958044     773 kubelet.go:2211] "Container runtime network not ready" networkReady="NetworkReady=false reason:NetworkPluginNotReady message:Network plugin returns error: cni plugin not initialized"
Aug 17 17:51:16 config-control-plane kubelet[773]: I0817 17:51:16.495632     773 topology_manager.go:187] "Topology Admit Handler"
Aug 17 17:51:16 config-control-plane kubelet[773]: W0817 17:51:16.503981     773 container.go:586] Failed to update stats for container "/kubelet/kubepods/besteffort/pod3c431678-6cc4-48de-b070-3076f8f49886": /sys/fs/cgroup/cpuset/kubelet/kubepods/besteffort/pod3c431678-6cc4-48de-b070-3076f8f49886/cpuset.mems found to be empty, continuing to push stats
Aug 17 17:51:16 config-control-plane kubelet[773]: I0817 17:51:16.616052     773 reconciler.go:224] "operationExecutor.VerifyControllerAttachedVolume started for volume \"tmp-dir\" (UniqueName: \"kubernetes.io/empty-dir/3c431678-6cc4-48de-b070-3076f8f49886-tmp-dir\") pod \"metrics-server-99b5b5fc6-89vcp\" (UID: \"3c431678-6cc4-48de-b070-3076f8f49886\") "
Aug 17 17:51:16 config-control-plane kubelet[773]: I0817 17:51:16.616147     773 reconciler.go:224] "operationExecutor.VerifyControllerAttachedVolume started for volume \"kube-api-access-c9tqh\" (UniqueName: \"kubernetes.io/projected/3c431678-6cc4-48de-b070-3076f8f49886-kube-api-access-c9tqh\") pod \"metrics-server-99b5b5fc6-89vcp\" (UID: \"3c431678-6cc4-48de-b070-3076f8f49886\") "
Aug 17 17:51:18 config-control-plane kubelet[773]: I0817 17:51:18.479566     773 topology_manager.go:187] "Topology Admit Handler"
Aug 17 17:51:18 config-control-plane kubelet[773]: I0817 17:51:18.488778     773 topology_manager.go:187] "Topology Admit Handler"
Aug 17 17:51:18 config-control-plane kubelet[773]: W0817 17:51:18.491169     773 container.go:586] Failed to update stats for container "/kubelet/kubepods/pod52c3613d-62d6-44d1-9f6a-19a247675334": /sys/fs/cgroup/cpuset/kubelet/kubepods/pod52c3613d-62d6-44d1-9f6a-19a247675334/cpuset.mems found to be empty, continuing to push stats
Aug 17 17:51:18 config-control-plane kubelet[773]: I0817 17:51:18.643932     773 reconciler.go:224] "operationExecutor.VerifyControllerAttachedVolume started for volume \"kube-api-access-sk77j\" (UniqueName: \"kubernetes.io/projected/92338b8a-af32-45ce-8373-9d7dbdadfe5b-kube-api-access-sk77j\") pod \"controller-fb659dc8-sqvlg\" (UID: \"92338b8a-af32-45ce-8373-9d7dbdadfe5b\") "
Aug 17 17:51:18 config-control-plane kubelet[773]: I0817 17:51:18.644068     773 reconciler.go:224] "operationExecutor.VerifyControllerAttachedVolume started for volume \"kube-api-access-dn42l\" (UniqueName: \"kubernetes.io/projected/52c3613d-62d6-44d1-9f6a-19a247675334-kube-api-access-dn42l\") pod \"speaker-qzsn6\" (UID: \"52c3613d-62d6-44d1-9f6a-19a247675334\") "
Aug 17 17:51:18 config-control-plane kubelet[773]: I0817 17:51:18.956913     773 topology_manager.go:187] "Topology Admit Handler"
Aug 17 17:51:18 config-control-plane kubelet[773]: I0817 17:51:18.960215     773 topology_manager.go:187] "Topology Admit Handler"
Aug 17 17:51:18 config-control-plane kubelet[773]: I0817 17:51:18.960466     773 topology_manager.go:187] "Topology Admit Handler"
Aug 17 17:51:19 config-control-plane kubelet[773]: I0817 17:51:19.150079     773 reconciler.go:224] "operationExecutor.VerifyControllerAttachedVolume started for volume \"kube-api-access-zdbfs\" (UniqueName: \"kubernetes.io/projected/8dc301c9-f72b-496a-b3e3-0839ad1ae4ac-kube-api-access-zdbfs\") pod \"local-path-provisioner-547f784dff-fpkm8\" (UID: \"8dc301c9-f72b-496a-b3e3-0839ad1ae4ac\") "
Aug 17 17:51:19 config-control-plane kubelet[773]: I0817 17:51:19.150187     773 reconciler.go:224] "operationExecutor.VerifyControllerAttachedVolume started for volume \"config-volume\" (UniqueName: \"kubernetes.io/configmap/0164a305-40c5-4f77-b1e0-da7eef569f37-config-volume\") pod \"coredns-558bd4d5db-dsndw\" (UID: \"0164a305-40c5-4f77-b1e0-da7eef569f37\") "
Aug 17 17:51:19 config-control-plane kubelet[773]: I0817 17:51:19.150237     773 reconciler.go:224] "operationExecutor.VerifyControllerAttachedVolume started for volume \"kube-api-access-n9ccw\" (UniqueName: \"kubernetes.io/projected/0164a305-40c5-4f77-b1e0-da7eef569f37-kube-api-access-n9ccw\") pod \"coredns-558bd4d5db-dsndw\" (UID: \"0164a305-40c5-4f77-b1e0-da7eef569f37\") "
Aug 17 17:51:19 config-control-plane kubelet[773]: I0817 17:51:19.150435     773 reconciler.go:224] "operationExecutor.VerifyControllerAttachedVolume started for volume \"config-volume\" (UniqueName: \"kubernetes.io/configmap/8dc301c9-f72b-496a-b3e3-0839ad1ae4ac-config-volume\") pod \"local-path-provisioner-547f784dff-fpkm8\" (UID: \"8dc301c9-f72b-496a-b3e3-0839ad1ae4ac\") "
Aug 17 17:51:19 config-control-plane kubelet[773]: I0817 17:51:19.150581     773 reconciler.go:224] "operationExecutor.VerifyControllerAttachedVolume started for volume \"kube-api-access-gks5d\" (UniqueName: \"kubernetes.io/projected/0ee912e3-fb21-4f32-8661-ff4a58e07ea5-kube-api-access-gks5d\") pod \"coredns-558bd4d5db-bjv9d\" (UID: \"0ee912e3-fb21-4f32-8661-ff4a58e07ea5\") "
Aug 17 17:51:19 config-control-plane kubelet[773]: I0817 17:51:19.150717     773 reconciler.go:224] "operationExecutor.VerifyControllerAttachedVolume started for volume \"config-volume\" (UniqueName: \"kubernetes.io/configmap/0ee912e3-fb21-4f32-8661-ff4a58e07ea5-config-volume\") pod \"coredns-558bd4d5db-bjv9d\" (UID: \"0ee912e3-fb21-4f32-8661-ff4a58e07ea5\") "
Aug 17 17:51:25 config-control-plane kubelet[773]: E0817 17:51:25.249709     773 cadvisor_stats_provider.go:415] "Partial failure issuing cadvisor.ContainerInfoV2" err="partial failures: [\"/kubelet/kubepods/pod52c3613d-62d6-44d1-9f6a-19a247675334\": RecentStats: unable to find data in memory cache], [\"/kubelet/kubepods/besteffort/pod3c431678-6cc4-48de-b070-3076f8f49886\": RecentStats: unable to find data in memory cache]"
Aug 17 17:51:26 config-control-plane kubelet[773]: E0817 17:51:26.277464     773 cadvisor_stats_provider.go:415] "Partial failure issuing cadvisor.ContainerInfoV2" err="partial failures: [\"/kubelet/kubepods/pod52c3613d-62d6-44d1-9f6a-19a247675334\": RecentStats: unable to find data in memory cache], [\"/kubelet/kubepods/besteffort/pod3c431678-6cc4-48de-b070-3076f8f49886\": RecentStats: unable to find data in memory cache]"
